src/irr/asset/CGLSLBRDFBuiltinIncludeLoader.h

```diff
@@ -447,6 +491,35 @@ float irr_glsl_ggx_smith_height_correlated_aniso_wo_numerator(in float at, in fl
     return 0.5 / (Vterm + Lterm);
 }
 
+
+float _C(in float NdotX, in float NdotX2, in float a)
+{
+    return NdotX / (a * sqrt(1.0 - NdotX2));
+}
+//_C squared
+float _C2(in float NdotX2, in float a2)
+{
+    return NdotX2 / (a2 * (1.0 - NdotX2));
+}
+//G1 = 1/(1+_Lambda)
+float _Lambda(in float c, in float c2)
+{
+    float nom = 0.396*c2 - 1.259*c + 1.0;
+    float denom = 2.181*c2 + 3.535*c;
+
+    //actually i think we could get rid of mix() since nom/denom is almost constant for c>1.6 (i.e. is going down but very slowly, at c=20 it's ~0.9)
+    return mix(c<1.6, 1.0, nom/denom);
+}
+//i wonder where i got irr_glsl_ggx_smith_height_correlated() from because it looks very different from 1/(1+L_v+L_l) form
```

Comment:
we factored out the denominator from the denominator, like you should do here (albeit not by hand, use SageMath to simplify `1.0/((1.0+L_v+L_l)*4.0*params.NdotV)` in terms of `NdotV` and `NdotL`, so we have a `irr_glsl_ggx_smith_height_correlated_wo_numerator`)

Reply:
well, i guess sage is bad at this XD
for this
```
sage: C(d,a) = d/(a*sqrt(1.0-d*d))
sage: L(c) = (0.396*c*c - 1.259*c + 1.0)/(2.181*c*c + 3.535*c)
sage: bs = 1.0/(1.0 + L((C(nv,a)) + L(C(nl,a))) * 4.0 * nv)
bs.full_simplify()
```
i get
```
((-340.5754492056132*a^10 + 3033.953424711557*a^8 + 2077.084913817077*a^6 + 129.6114934600012*a^4 - 76.820398842161*a^2)*nl^32 + (5449.207187289811*a^10 - 45509.30137067335*a^8 - 29079.18879343909*a^6 - 1684.949414980016*a^4 + 921.844786105932*a^2)*nl^30 + (-40869.05390467359*a^10 + 318565.1095947134*a^8 + 189014.7271573541*a^6 + 10109.69648988009*a^4 - 5070.146323582626*a^2)*nl^28 + (190722.2515551434*a^10 - 1380448.808243758*a^8 - 756058.9086294165*a^6 - 37068.88712956033*a^4 + 16900.48774527542*a^2)*nl^26 + (-619847.317554216*a^10 + 4141346.424731274*a^8 + 2079161.998730895*a^6 + 92672.21782390073*a^4 - 38026.0974268697*a^2)*nl^24 + (1487633.562130118*a^10 - 9110962.134408802*a^8 - 4158323.997461791*a^6 - 166809.9920830213*a^4 + 60841.75588299151*a^2)*nl^22 + (-2727328.19723855*a^10 + 15184936.8906813*a^8 + 6237485.996192686*a^6 + 222413.3227773617*a^4 - 70982.04853015675*a^2)*nl^20 + (3896183.138912215*a^10 - 19523490.2880189*a^8 - 7128555.424220213*a^6 - 222413.3227773616*a^4 + 60841.7558829915*a^2)*nl^18 + (-4383206.031276241*a^10 + 19523490.2880189*a^8 + 6237485.996192687*a^6 + 166809.9920830212*a^4 - 38026.0974268697*a^2)*nl^16 + (3896183.138912214*a^10 - 15184936.8906813*a^8 - 4158323.997461791*a^6 - 92672.2178239007*a^4 + 16900.48774527542*a^2)*nl^14 + (-2727328.19723855*a^10 + 9110962.134408802*a^8 + 2079161.998730895*a^6 + 37068.8871295603*a^4 - 5070.146323582626*a^2)*nl^12 + (1487633.562130118*a^10 - 4141346.424731274*a^8 - 756058.9086294164*a^6 - 10109.69648988009*a^4 + 921.844786105932*a^2)*nl^10 - 340.5754492056132*a^10 + (-619847.3175542159*a^10 + 1380448.808243758*a^8 + 189014.7271573541*a^6 + 1684.949414980016*a^4 - 76.820398842161*a^2)*nl^8 + (190722.2515551434*a^10 - 318565.1095947134*a^8 - 29079.18879343909*a^6 - 129.6114934600012*a^4)*nl^6 + ((340.5754492056132*a^10 - 3033.953424711557*a^8 + 2178.822528932337*a^6 - 24430.15170854113*a^4 + 9326.962899799812*a^2 - 234.7413830603392)*nl^32 + (-5449.207187289811*a^10 + 45509.30137067335*a^8 - 34759.42284780213*a^6 + 341892.5124261158*a^4 - 121173.6972985554*a^2 + 2816.896596724071)*nl^30 + (40869.05390467359*a^10 - 318565.1095947134*a^8 + 257855.5543313344*a^6 - 2221458.856062263*a^4 + 726581.2613982793*a^2 - 15492.93128198239)*nl^28 + (-190722.2515551434*a^10 + 1380448.808243758*a^8 - 1180378.977821567*a^6 + 8882465.52541909*a^4 - 2662441.243019164*a^2 + 51643.10427327463)*nl^26 + (619847.3175542159*a^10 - 4141346.424731274*a^8 + 3730151.660622055*a^6 - 24417512.9731201*a^4 + 6651877.98561159*a^2 - 116196.9846148679)*nl^24 + (-1487633.562130118*a^10 + 9110962.134408802*a^8 - 8622166.053114701*a^6 + 48816491.5026754*a^4 - 11965775.1546155*a^2 + 185915.1753837886)*nl^22 + (2727328.19723855*a^10 - 15184936.8906813*a^8 + 15063330.7547681*a^6 - 73196935.588666*a^4 + 15944226.5801735*a^2 - 216901.0379477534)*nl^20 + (-3896183.138912215*a^10 + 19523490.2880189*a^8 - 20258208.9698723*a^6 + 83621867.3409358*a^4 - 15934086.2875263*a^2 + 185915.1753837886)*nl^18 + (4383206.031276242*a^10 - 19523490.2880189*a^8 + 21149278.3978998*a^6 - 73141332.2579717*a^4 + 11942959.4961594*a^2 - 116196.9846148679)*nl^16 + (-3896183.138912215*a^10 + 15184936.8906813*a^8 - 17142492.753499*a^6 + 48742353.7284163*a^4 - 6630752.375929995*a^2 + 51643.10427327463)*nl^14 + (2727328.19723855*a^10 - 9110962.134408802*a^8 + 10701328.0518456*a^6 - 24361909.6424258*a^4 + 2650610.901597471*a^2 - 15492.93128198239)*nl^12 + (-1487633.562130118*a^10 + 4141346.424731274*a^8 - 5053254.750723533*a^6 + 8855506.334779412*a^4 - 722432.9598608026*a^2 + 2816.896596724071)*nl^10 + 340.5754492056132*a^10 + (619847.317554216*a^10 - 1380448.808243758*a^8 + 1747423.159293629*a^6 - 2213034.108987363*a^4 + 120328.6729112916*a^2 - 234.7413830603392)*nl^8 + (-190722.2515551434*a^10 + 318565.1095947134*a^8 - 417791.0926952494*a^6 + 340337.1745045958*a^4 - 9250.14250095765*a^2)*nl^6 + (40869.05390467359*a^10 - 45509.30137067335*a^8 + 61761.52672742413*a^6 - 24300.54021508113*a^4)*nl^4 + (-5449.207187289811*a^10 + 3033.953424711557*a^8 - 4255.907442749414*a^6)*nl^2)*nv^6 + (-40869.05390467359*a^10 + 45509.30137067335*a^8 + 2077.084913817077*a^6)*nl^4 + ((-1021.72634761684*a^10 + 9101.860274134671*a^8 - 2280.560144047596*a^6 + 48989.91491054227*a^4 - 18730.74619844178*a^2 + 469.4827661206784)*nl^32 + (16347.62156186943*a^10 - 136527.90411202*a^8 + 40439.65690216516*a^6 - 685469.9742672116*a^4 + 243269.2393832167*a^2 - 5633.793193448141)*nl^30 + (-122607.1617140208*a^10 + 955695.3287841403*a^8 - 326696.3815053147*a^6 + 4453027.408614405*a^4 - 1458232.669120141*a^2 + 30985.86256396477)*nl^28 + (572166.7546654302*a^10 - 4141346.424731275*a^8 + 1604699.047013718*a^6 - 17801999.9379677*a^4 + 5341782.973783603*a^2 - 103286.2085465493)*nl^26 + (-1859541.952662648*a^10 + 12424039.2741938*a^8 - 5381141.322513215*a^6 + 48927698.1640641*a^4 - 13341782.0686501*a^2 + 232393.9692297358)*nl^24 + (4462900.686390355*a^10 - 27332886.4032264*a^8 + 13086008.1087676*a^6 - 97799792.9974339*a^4 + 23992392.065114*a^2 - 371830.3507675772)*nl^22 + (-8181984.591715651*a^10 + 45554810.672044*a^8 - 23889175.5133436*a^6 + 146616284.500109*a^4 - 31959435.2088771*a^2 + 433802.0758955069)*nl^20 + (11688549.4167366*a^10 - 58570470.8640566*a^8 + 33387862.5155243*a^6 - 167466148.004649*a^4 + 31929014.3309356*a^2 - 371830.3507675773)*nl^18 + (-13149618.0938287*a^10 + 58570470.8640566*a^8 - 36061070.7996069*a^6 + 146449474.508026*a^4 - 23923945.0897456*a^2 + 232393.9692297358)*nl^16 + (11688549.4167366*a^10 - 45554810.672044*a^8 + 30126661.5095363*a^6 - 97577379.6746565*a^4 + 13278405.2396053*a^2 - 103286.2085465493)*nl^14 + (-8181984.591715651*a^10 + 27332886.4032264*a^8 - 19323494.1049603*a^6 + 48760888.1719811*a^4 - 5306291.949518524*a^2 + 30985.86256396478)*nl^12 + (4462900.686390355*a^10 - 12424039.2741938*a^8 + 9350450.592817651*a^6 - 17721122.3660487*a^4 + 1445787.764507711*a^2 - 5633.793193448141)*nl^10 - 1021.72634761684*a^10 + (-1859541.952662648*a^10 + 4141346.424731275*a^8 - 3305831.591429905*a^6 + 4427753.167389706*a^4 - 240734.1662214254*a^2 + 469.4827661206784)*nl^8 + (572166.7546654302*a^10 - 955695.3287841403*a^8 + 806502.9965970598*a^6 - 680803.9605026516*a^4 + 18500.2850019153*a^2)*nl^6 + (-122607.1617140208*a^10 + 136527.90411202*a^8 - 121445.9685410312*a^6 + 48601.08043016226*a^4)*nl^4 + (16347.62156186943*a^10 - 9101.860274134671*a^8 + 8511.814885498829*a^6)*nl^2)*nv^4 + (5449.207187289811*a^10 - 3033.953424711557*a^8)*nl^2 + ((1021.72634761684*a^10 - 9101.860274134671*a^8 - 1975.347298701818*a^6 - 24689.37469546113*a^4 + 9480.603697484134*a^2 - 234.7413830603392)*nl^32 + (-16347.62156186943*a^10 + 136527.90411202*a^8 + 23398.95473907605*a^6 + 345262.4112560758*a^4 - 123017.3868707672*a^2 + 2816.896596724071)*nl^30 + (122607.1617140208*a^10 - 955695.3287841403*a^8 - 120173.8999833738*a^6 - 2241678.249042023*a^4 + 736721.5540454446*a^2 - 15492.93128198239)*nl^28 + (-572166.7546654302*a^10 + 4141346.424731275*a^8 + 331738.8394372659*a^6 + 8956603.299678212*a^4 - 2696242.218509714*a^2 + 51643.10427327463)*nl^26 + (1859541.952662648*a^10 - 12424039.2741938*a^8 - 428172.3368397355*a^6 - 24602857.4087679*a^4 + 6727930.180465329*a^2 - 116196.9846148679)*nl^24 + (-4462900.686390355*a^10 + 27332886.4032264*a^8 - 305518.0581911202*a^6 + 49150111.4868415*a^4 - 12087458.6663815*a^2 + 185915.1753837886)*nl^22 + (8181984.591715651*a^10 - 45554810.672044*a^8 + 2588358.762382761*a^6 - 73641762.2342207*a^4 + 16086190.6772338*a^2 - 216901.0379477534)*nl^20 + (-11688549.4167366*a^10 + 58570470.8640566*a^8 - 6001098.121431838*a^6 + 84066693.9864905*a^4 - 16055769.7992923*a^2 + 185915.1753837886)*nl^18 + (13149618.0938287*a^10 - 58570470.8640566*a^8 + 8674306.40551442*a^6 - 73474952.2421377*a^4 + 12019011.6910131*a^2 - 116196.9846148679)*nl^16 + (-11688549.4167366*a^10 + 45554810.672044*a^8 - 8825844.758575443*a^6 + 48927698.1640641*a^4 - 6664553.351420546*a^2 + 51643.10427327463)*nl^14 + (8181984.591715651*a^10 - 27332886.4032264*a^8 + 6543004.054383802*a^6 - 24436047.4166849*a^4 + 2660751.194244636*a^2 - 15492.93128198239)*nl^12 + (-4462900.686390355*a^10 + 12424039.2741938*a^8 - 3541136.933464701*a^6 + 8875725.727759171*a^4 - 724276.6494330144*a^2 + 2816.896596724071)*nl^10 + 1021.72634761684*a^10 + (1859541.952662648*a^10 - 4141346.424731275*a^8 + 1369393.704978921*a^6 - 2216404.007817323*a^4 + 120482.3137089759*a^2 - 234.7413830603392)*nl^8 + (-572166.7546654302*a^10 + 955695.3287841403*a^8 - 359632.7151083713*a^6 + 340596.3974915158*a^4 - 9250.14250095765*a^2)*nl^6 + (122607.1617140208*a^10 - 136527.9041120201*a^8 + 57607.35689978999*a^6 - 24300.54021508113*a^4)*nl^4 + (-16347.62156186943*a^10 + 9101.860274134671*a^8 - 4255.907442749414*a^6)*nl^2)*nv^2 + ((1934.29061997787*a^9 - 641.5802369130929*a^7 - 1422.133250031137*a^5 - 341.1774975656799*a^3)*nl^31 + (-29014.35929966805*a^9 + 8982.123316783296*a^7 + 18487.73225040478*a^5 + 4094.12997078816*a^3)*nl^29 + (203100.5150976763*a^9 - 58383.80155909143*a^7 - 110926.3935024287*a^5 - 22517.71483933488*a^3)*nl^27 + (-880102.2320899307*a^9 + 233535.2062363656*a^7 + 406730.1095089052*a^5 + 75059.04946444959*a^3)*nl^25 + (2640306.696269792*a^9 - 642221.8171500053*a^7 - 1016825.273772263*a^5 - 168882.8612950116*a^3)*nl^23 + (-5808674.731793541*a^9 + 1284443.63430001*a^7 + 1830285.492790074*a^5 + 270212.5780720185*a^3)*nl^21 + (9681124.552989237*a^9 - 1926665.451450016*a^7 - 2440380.657053431*a^5 - 315248.0077506883*a^3)*nl^19 + (-12447160.1395576*a^9 + 2201903.373085732*a^7 + 2440380.657053431*a^5 + 270212.5780720185*a^3)*nl^17 + (12447160.1395576*a^9 - 1926665.451450016*a^7 - 1830285.492790074*a^5 - 168882.8612950116*a^3)*nl^15 + (-9681124.552989237*a^9 + 1284443.63430001*a^7 + 1016825.273772263*a^5 + 75059.04946444958*a^3)*nl^13 + (5808674.731793541*a^9 - 642221.8171500053*a^7 - 406730.1095089053*a^5 - 22517.71483933488*a^3)*nl^11 - 1934.29061997787*a^9*nl + (-2640306.696269792*a^9 + 233535.2062363655*a^7 + 110926.3935024287*a^5 + 4094.12997078816*a^3)*nl^9 + (880102.2320899306*a^9 - 58383.80155909141*a^7 - 18487.73225040478*a^5 - 341.1774975656799*a^3)*nl^7 + ((-1934.29061997787*a^9 + 641.5802369130929*a^7 - 14332.54986052582*a^5 + 20331.55117379083*a^3 - 2282.835733475376*a)*nl^31 + (29014.35929966805*a^9 - 8982.123316783296*a^7 + 202077.8312973925*a^5 - 263968.9877617152*a^3 + 27394.02880170451*a)*nl^29 + (-203100.5150976763*a^9 + 58383.80155909141*a^7 - 1322749.769558254*a^5 + 1581766.861584897*a^3 - 150667.1584093748*a)*nl^27 + (880102.2320899306*a^9 - 233535.2062363656*a^7 + 5327974.542733826*a^5 - 5792305.920864844*a^3 + 502223.8613645827*a)*nl^25 + (-2640306.696269792*a^9 + 642221.8171500053*a^7 - 14753612.5198952*a^5 + 14462000.039796*a^3 - 1130003.688070311*a)*nl^23 + (5808674.731793541*a^9 - 1284443.63430001*a^7 + 29710590.0945449*a^5 - 25997823.4993738*a^3 + 1808005.900912497*a)*nl^21 + (-9681124.552989237*a^9 + 1926665.451450016*a^7 - 44870932.7239491*a^5 + 34618729.2361531*a^3 - 2109340.217731247*a)*nl^19 + (12447160.1395576*a^9 - 2201903.373085732*a^7 + 51629691.778378*a^5 - 34573693.8064744*a^3 + 1808005.900912498*a)*nl^17 + (-12447160.1395576*a^9 + 1926665.451450016*a^7 - 45481027.8882124*a^5 + 25896493.7825968*a^3 - 1130003.688070311*a)*nl^15 + (9681124.552989237*a^9 - 1284443.63430001*a^7 + 30524050.3135628*a^5 - 14368176.2279654*a^3 + 502223.8613645827*a)*nl^13 + (-5808674.731793542*a^9 + 642221.8171500053*a^7 - 15363707.6841586*a^5 + 5739764.586239729*a^3 - 150667.1584093748*a)*nl^11 + 1934.29061997787*a^9*nl + (2640306.696269792*a^9 - 233535.2062363655*a^7 + 5623778.258740302*a^5 - 1563343.27671635*a^3 + 27394.02880170451*a)*nl^9 + (-880102.2320899307*a^9 + 58383.80155909141*a^7 - 1415188.430810278*a^5 + 260216.0352884927*a^3 - 2282.835733475376*a)*nl^7 + (203100.5150976763*a^9 - 8982.123316783298*a^7 + 219143.4302977662*a^5 - 19990.37367622516*a^3)*nl^5 + (-29014.35929966805*a^9 + 641.5802369130929*a^7 - 15754.68311055695*a^5)*nl^3)*nv^6 + (-203100.5150976763*a^9 + 8982.123316783296*a^7 + 1422.133250031137*a^5)*nl^5 + ((5802.871859933609*a^9 - 1924.740710739279*a^7 + 27242.9664710205*a^5 - 41004.27984514735*a^3 + 4565.671466950752*a)*nl^31 + (-87043.07789900413*a^9 + 26946.36995034989*a^7 - 385667.9303443803*a^5 + 532032.1054942185*a^3 - 54788.05760340902*a)*nl^29 + (609301.5452930289*a^9 - 175151.4046772742*a^7 + 2534573.14561408*a^5 - 3186051.438009129*a^3 + 301334.3168187496*a)*nl^27 + (-2640306.696269792*a^9 + 700605.6187090968*a^7 - 10249218.9759587*a^5 + 11659670.8911941*a^3 - 1004447.722729165*a)*nl^25 + (7920920.088809375*a^9 - 1926665.451450016*a^7 + 28490399.7660182*a^5 - 29092882.940887*a^3 + 2260007.376140622*a)*nl^23 + (-17426024.1953806*a^9 + 3853330.902900031*a^7 - 57590894.6962998*a^5 + 52265859.5768196*a^3 - 3616011.801824995*a)*nl^21 + (29043373.6589677*a^9 - 5779996.354350047*a^7 + 87301484.7908448*a^5 - 69552706.4800568*a^3 + 4218680.435462494*a)*nl^19 + (-37341480.4186728*a^9 + 6605710.119257197*a^7 - 100819002.899703*a^5 + 69417600.1910208*a^3 - 3616011.801824995*a)*nl^17 + (37341480.4186728*a^9 - 5779996.354350047*a^7 + 89131770.2836348*a^5 - 51961870.4264886*a^3 + 2260007.376140622*a)*nl^15 + (-29043373.6589677*a^9 + 3853330.902900031*a^7 - 60031275.3533533*a^5 + 28811411.5053953*a^3 - 1004447.722729165*a)*nl^13 + (17426024.1953806*a^9 - 1926665.451450016*a^7 + 30320685.2588083*a^5 - 11502046.8873188*a^3 + 301334.3168187496*a)*nl^11 - 5802.871859933609*a^9*nl + (-7920920.088809375*a^9 + 700605.6187090967*a^7 - 11136630.1239782*a^5 + 3130780.683403488*a^3 - 54788.05760340902*a)*nl^9 + (2640306.696269792*a^9 - 175151.4046772742*a^7 + 2811889.129370151*a^5 - 520773.2480745511*a^3 + 4565.671466950752*a)*nl^7 + (-609301.5452930289*a^9 + 26946.36995034989*a^7 - 436864.7273455013*a^5 + 39980.74735245031*a^3)*nl^5 + (87043.07789900413*a^9 - 1924.740710739279*a^7 + 31509.36622111391*a^5)*nl^3)*nv^4 + (29014.35929966805*a^9 - 641.5802369130929*a^7)*nl^3 + ((-5802.871859933609*a^9 + 1924.740710739279*a^7 - 11488.28336046355*a^5 + 21013.90616892219*a^3 - 2282.835733475376*a)*nl^31 + (87043.07789900413*a^9 - 26946.36995034989*a^7 + 165102.366796583*a^5 - 272157.2477032915*a^3 + 27394.02880170451*a)*nl^29 + (-609301.5452930289*a^9 + 175151.4046772743*a^7 - 1100896.982553397*a^5 + 1626802.291263567*a^3 - 150667.1584093748*a)*nl^27 + (2640306.696269792*a^9 - 700605.6187090968*a^7 + 4514514.323716015*a^5 - 5942424.019793743*a^3 + 502223.8613645827*a)*nl^25 + (-7920920.088809375*a^9 + 1926665.451450016*a^7 - 12719961.9723507*a^5 + 14799765.762386*a^3 - 1130003.688070311*a)*nl^23 + (17426024.1953806*a^9 - 3853330.902900031*a^7 + 26050019.1089648*a^5 - 26538248.6555178*a^3 + 1808005.900912497*a)*nl^21 + (-29043373.6589677*a^9 + 5779996.354350047*a^7 - 39990171.4098422*a^5 + 35249225.2516544*a^3 - 2109340.217731247*a)*nl^19 + (37341480.4186728*a^9 - 6605710.119257197*a^7 + 46748930.4642712*a^5 - 35114118.9626184*a^3 + 1808005.900912498*a)*nl^17 + (-37341480.4186728*a^9 + 5779996.354350047*a^7 - 41820456.9026323*a^5 + 26234259.5051868*a^3 - 1130003.688070311*a)*nl^15 + (29043373.6589677*a^9 - 3853330.902900031*a^7 + 28490399.7660182*a^5 - 14518294.3268943*a^3 + 502223.8613645827*a)*nl^13 + (-17426024.1953806*a^9 + 1926665.451450016*a^7 - 14550247.4651408*a^5 + 5784800.015918399*a^3 - 150667.1584093748*a)*nl^11 + 5802.871859933609*a^9*nl + (7920920.088809375*a^9 - 700605.6187090967*a^7 + 5401925.471735444*a^5 - 1571531.536657927*a^3 + 27394.02880170451*a)*nl^9 + (-2640306.696269792*a^9 + 175151.4046772742*a^7 - 1378212.966309468*a^5 + 260898.3902836241*a^3 - 2282.835733475376*a)*nl^7 + (609301.5452930289*a^9 - 26946.36995034989*a^7 + 216299.1637977039*a^5 - 19990.37367622516*a^3)*nl^5 + (-87043.07789900413*a^9 + 1924.740710739279*a^7 - 15754.68311055695*a^5)*nl^3)*nv^2)*sqrt(-1.0*nl^2 + 1.0) + (((11294.48766992642*a^7 - 36443.55577869599*a^5 + 15251.10565494534*a^3 - 465.7157104548775*a)*nl^32 + (-169417.3150488964*a^7 + 510209.7809017438*a^5 - 198264.3735142894*a^3 + 5588.58852545853*a)*nl^30 + (1185921.205342275*a^7 - 3316363.575861335*a^5 + 1189586.241085737*a^3 - 30737.23689002192*a)*nl^28 + (-5138991.889816523*a^7 + 13265454.3034453*a^5 - 4361816.217314367*a^3 + 102457.456300073*a)*nl^26 + (15416975.6694496*a^7 - 36479999.3344747*a^5 + 10904540.5432859*a^3 - 230529.2766751644*a)*nl^24 + (-33917346.472789*a^7 + 72959998.6689494*a^5 - 19628172.9779146*a^3 + 368846.842680263*a)*nl^22 + (56528910.7879817*a^7 - 109439998.003424*a^5 + 26170897.3038862*a^3 - 430321.3164603068*a)*nl^20 + (-72680028.1559765*a^7 + 125074283.432485*a^5 - 26170897.3038862*a^3 + 368846.842680263*a)*nl^18 + (72680028.1559765*a^7 - 109439998.003424*a^5 + 19628172.9779146*a^3 - 230529.2766751644*a)*nl^16 + (-56528910.7879817*a^7 + 72959998.6689494*a^5 - 10904540.5432859*a^3 + 102457.4563000731*a)*nl^14 + (33917346.472789*a^7 - 36479999.3344747*a^5 + 4361816.217314368*a^3 - 30737.23689002192*a)*nl^12 + (-15416975.6694496*a^7 + 13265454.3034453*a^5 - 1189586.241085737*a^3 + 5588.58852545853*a)*nl^10 - 11294.48766992642*a^7*nl^2 + (5138991.889816523*a^7 - 3316363.575861334*a^5 + 198264.3735142894*a^3 - 465.7157104548775*a)*nl^8 + (-1185921.205342274*a^7 + 510209.7809017438*a^5 - 15251.10565494534*a^3)*nl^6 + (169417.3150488964*a^7 - 36443.55577869599*a^5)*nl^4)*nv^5 + ((-22588.97533985285*a^7 + 72887.11155739198*a^5 - 30502.21130989068*a^3 + 931.431420909755*a)*nl^32 + (338834.6300977927*a^7 - 1020419.561803488*a^5 + 396528.7470285788*a^3 - 11177.17705091706*a)*nl^30 + (-2371842.410684549*a^7 + 6632727.15172267*a^5 - 2379172.482171473*a^3 + 61474.47378004383*a)*nl^28 + (10277983.779633*a^7 - 26530908.6068907*a^5 + 8723632.434628734*a^3 - 204914.9126001461*a)*nl^26 + (-30833951.3388991*a^7 + 72959998.6689494*a^5 - 21809081.0865718*a^3 + 461058.5533503287*a)*nl^24 + (67834692.9455781*a^7 - 145919997.337899*a^5 + 39256345.9558293*a^3 - 737693.685360526*a)*nl^22 + (-113057821.575963*a^7 + 218879996.006848*a^5 - 52341794.6077724*a^3 + 860642.6329206135*a)*nl^20 + (145360056.311953*a^7 - 250148566.864969*a^5 + 52341794.6077724*a^3 - 737693.685360526*a)*nl^18 + (-145360056.311953*a^7 + 218879996.006848*a^5 - 39256345.9558293*a^3 + 461058.5533503287*a)*nl^16 + (113057821.575963*a^7 - 145919997.337899*a^5 + 21809081.0865718*a^3 - 204914.9126001461*a)*nl^14 + (-67834692.9455781*a^7 + 72959998.6689494*a^5 - 8723632.434628734*a^3 + 61474.47378004383*a)*nl^12 + (30833951.3388991*a^7 - 26530908.6068907*a^5 + 2379172.482171473*a^3 - 11177.17705091706*a)*nl^10 + 22588.97533985285*a^7*nl^2 + (-10277983.779633*a^7 + 6632727.15172267*a^5 - 396528.7470285788*a^3 + 931.431420909755*a)*nl^8 + (2371842.410684549*a^7 - 1020419.561803488*a^5 + 30502.21130989068*a^3)*nl^6 + (-338834.6300977927*a^7 + 72887.11155739198*a^5)*nl^4)*nv^3 + ((11294.48766992642*a^7 - 36443.55577869599*a^5 + 15251.10565494534*a^3 - 465.7157104548775*a)*nl^32 + (-169417.3150488964*a^7 + 510209.7809017438*a^5 - 198264.3735142894*a^3 + 5588.58852545853*a)*nl^30 + (1185921.205342274*a^7 - 3316363.575861335*a^5 + 1189586.241085737*a^3 - 30737.23689002192*a)*nl^28 + (-5138991.889816523*a^7 + 13265454.3034453*a^5 - 4361816.217314368*a^3 + 102457.456300073*a)*nl^26 + (15416975.6694496*a^7 - 36479999.3344747*a^5 + 10904540.5432859*a^3 - 230529.2766751644*a)*nl^24 + (-33917346.472789*a^7 + 72959998.6689494*a^5 - 19628172.9779146*a^3 + 368846.842680263*a)*nl^22 + (56528910.7879817*a^7 - 109439998.003424*a^5 + 26170897.3038862*a^3 - 430321.3164603068*a)*nl^20 + (-72680028.1559765*a^7 + 125074283.432485*a^5 - 26170897.3038862*a^3 + 368846.842680263*a)*nl^18 + (72680028.1559765*a^7 - 109439998.003424*a^5 + 19628172.9779146*a^3 - 230529.2766751644*a)*nl^16 + (-56528910.7879817*a^7 + 72959998.6689494*a^5 - 10904540.5432859*a^3 + 102457.4563000731*a)*nl^14 + (33917346.472789*a^7 - 36479999.3344747*a^5 + 4361816.217314367*a^3 - 30737.23689002192*a)*nl^12 + (-15416975.6694496*a^7 + 13265454.3034453*a^5 - 1189586.241085737*a^3 + 5588.58852545853*a)*nl^10 - 11294.48766992642*a^7*nl^2 + (5138991.889816523*a^7 - 3316363.575861335*a^5 + 198264.3735142894*a^3 - 465.7157104548775*a)*nl^8 + (-1185921.205342275*a^7 + 510209.7809017438*a^5 - 15251.10565494534*a^3)*nl^6 + (169417.3150488964*a^7 - 36443.55577869599*a^5)*nl^4)*nv + (((2407.868425883686*a^8 - 26302.86255025803*a^6 + 30655.1945279348*a^4 - 4119.861118470091*a^2)*nl^31 + (-36118.02638825528*a^8 + 368240.0757036123*a^6 - 398517.5288631524*a^4 + 49438.33342164109*a^2)*nl^29 + (252826.184717787*a^8 - 2393560.49207348*a^6 + 2391105.173178914*a^4 - 271910.833819026*a^2)*nl^27 + (-1095580.133777077*a^8 + 9574241.96829392*a^6 - 8767385.634989353*a^4 + 906369.4460634199*a^2)*nl^25 + (3286740.401331231*a^8 - 26329165.4128083*a^6 + 21918464.0874734*a^4 - 2039331.253642695*a^2)*nl^23 + (-7230828.882928707*a^8 + 52658330.8256165*a^6 - 39453235.3574521*a^4 + 3262930.005828312*a^2)*nl^21 + (12051381.4715479*a^8 - 78987496.2384248*a^6 + 52604313.8099361*a^4 - 3806751.673466364*a^2)*nl^19 + (-15494633.3205615*a^8 + 90271424.2724855*a^6 - 52604313.8099361*a^4 + 3262930.005828312*a^2)*nl^17 + (15494633.3205615*a^8 - 78987496.2384248*a^6 + 39453235.3574521*a^4 - 2039331.253642695*a^2)*nl^15 + (-12051381.4715478*a^8 + 52658330.8256165*a^6 - 21918464.0874734*a^4 + 906369.44606342*a^2)*nl^13 + (7230828.882928707*a^8 - 26329165.4128083*a^6 + 8767385.634989353*a^4 - 271910.833819026*a^2)*nl^11 + (-3286740.40133123*a^8 + 9574241.96829392*a^6 - 2391105.173178914*a^4 + 49438.33342164109*a^2)*nl^9 - 2407.868425883686*a^8*nl + (1095580.133777077*a^8 - 2393560.49207348*a^6 + 398517.5288631525*a^4 - 4119.861118470091*a^2)*nl^7 + (-252826.184717787*a^8 + 368240.0757036123*a^6 - 30655.1945279348*a^4)*nl^5 + (36118.02638825528*a^8 - 26302.86255025803*a^6)*nl^3)*nv^5 + ((-4815.736851767371*a^8 + 52605.72510051605*a^6 - 61310.3890558696*a^4 + 8239.722236940182*a^2)*nl^31 + (72236.05277651057*a^8 - 736480.1514072246*a^6 + 797035.0577263049*a^4 - 98876.66684328219*a^2)*nl^29 + (-505652.3694355739*a^8 + 4787120.98414696*a^6 - 4782210.346357829*a^4 + 543821.667638052*a^2)*nl^27 + (2191160.267554154*a^8 - 19148483.9365878*a^6 + 17534771.2699787*a^4 - 1812738.89212684*a^2)*nl^25 + (-6573480.802662461*a^8 + 52658330.8256165*a^6 - 43836928.1749468*a^4 + 4078662.50728539*a^2)*nl^23 + (14461657.7658574*a^8 - 105316661.651233*a^6 + 78906470.7149042*a^4 - 6525860.011656624*a^2)*nl^21 + (-24102762.9430957*a^8 + 157974992.47685*a^6 - 105208627.619872*a^4 + 7613503.346932728*a^2)*nl^19 + (30989266.641123*a^8 - 180542848.544971*a^6 + 105208627.619872*a^4 - 6525860.011656624*a^2)*nl^17 + (-30989266.641123*a^8 + 157974992.47685*a^6 - 78906470.7149042*a^4 + 4078662.50728539*a^2)*nl^15 + (24102762.9430957*a^8 - 105316661.651233*a^6 + 43836928.1749468*a^4 - 1812738.89212684*a^2)*nl^13 + (-14461657.7658574*a^8 + 52658330.8256165*a^6 - 17534771.2699787*a^4 + 543821.667638052*a^2)*nl^11 + (6573480.80266246*a^8 - 19148483.9365878*a^6 + 4782210.346357829*a^4 - 98876.66684328219*a^2)*nl^9 + 4815.736851767371*a^8*nl + (-2191160.267554154*a^8 + 4787120.98414696*a^6 - 797035.0577263049*a^4 + 8239.722236940182*a^2)*nl^7 + (505652.3694355739*a^8 - 736480.1514072246*a^6 + 61310.3890558696*a^4)*nl^5 + (-72236.05277651057*a^8 + 52605.72510051605*a^6)*nl^3)*nv^3 + ((2407.868425883686*a^8 - 26302.86255025803*a^6 + 30655.1945279348*a^4 - 4119.861118470091*a^2)*nl^31 + (-36118.02638825528*a^8 + 368240.0757036123*a^6 - 398517.5288631524*a^4 + 49438.33342164109*a^2)*nl^29 + (252826.184717787*a^8 - 2393560.49207348*a^6 + 2391105.173178914*a^4 - 271910.833819026*a^2)*nl^27 + (-1095580.133777077*a^8 + 9574241.96829392*a^6 - 8767385.634989355*a^4 + 906369.4460634199*a^2)*nl^25 + (3286740.401331231*a^8 - 26329165.4128083*a^6 + 21918464.0874734*a^4 - 2039331.253642695*a^2)*nl^23 + (-7230828.882928708*a^8 + 52658330.8256165*a^6 - 39453235.3574521*a^4 + 3262930.005828313*a^2)*nl^21 + (12051381.4715478*a^8 - 78987496.2384248*a^6 + 52604313.8099361*a^4 - 3806751.673466364*a^2)*nl^19 + (-15494633.3205615*a^8 + 90271424.2724855*a^6 - 52604313.8099361*a^4 + 3262930.005828312*a^2)*nl^17 + (15494633.3205615*a^8 - 78987496.2384248*a^6 + 39453235.3574521*a^4 - 2039331.253642695*a^2)*nl^15 + (-12051381.4715478*a^8 + 52658330.8256165*a^6 - 21918464.0874734*a^4 + 906369.44606342*a^2)*nl^13 + (7230828.882928707*a^8 - 26329165.4128083*a^6 + 8767385.634989353*a^4 - 271910.833819026*a^2)*nl^11 + (-3286740.401331231*a^8 + 9574241.96829392*a^6 - 2391105.173178914*a^4 + 49438.33342164109*a^2)*nl^9 - 2407.868425883686*a^8*nl + (1095580.133777077*a^8 - 2393560.49207348*a^6 + 398517.5288631524*a^4 - 4119.861118470091*a^2)*nl^7 + (-252826.184717787*a^8 + 368240.0757036123*a^6 - 30655.1945279348*a^4)*nl^5 + (36118.02638825528*a^8 - 26302.86255025803*a^6)*nl^3)*nv)*sqrt(-1.0*nl^2 + 1.0))*sqrt(-1.0*nv^2 + 1.0))/((-340.5754492056132*a^10 + 3033.953424711557*a^8 + 2077.084913817077*a^6 + 129.6114934600012*a^4 - 76.820398842161*a^2)*nl^32 + (5449.207187289811*a^10 - 45509.30137067335*a^8 - 29079.18879343909*a^6 - 1684.949414980016*a^4 + 921.844786105932*a^2)*nl^30 + (-40869.05390467359*a^10 + 318565.1095947134*a^8 + 189014.7271573541*a^6 + 10109.69648988009*a^4 - 5070.146323582626*a^2)*nl^28 + (190722.2515551434*a^10 - 1380448.808243758*a^8 - 756058.9086294165*a^6 - 37068.88712956033*a^4 + 16900.48774527542*a^2)*nl^26 + (-619847.317554216*a^10 + 4141346.424731274*a^8 + 2079161.998730895*a^6 + 92672.21782390073*a^4 - 38026.0974268697*a^2)*nl^24 + (1487633.562130118*a^10 - 9110962.134408802*a^8 - 4158323.997461791*a^6 - 166809.9920830213*a^4 + 60841.75588299151*a^2)*nl^22 + (-2727328.19723855*a^10 + 15184936.8906813*a^8 + 6237485.996192686*a^6 + 222413.3227773617*a^4 - 70982.04853015675*a^2)*nl^20 + (3896183.138912215*a^10 - 19523490.2880189*a^8 - 7128555.424220213*a^6 - 222413.3227773616*a^4 + 60841.7558829915*a^2)*nl^18 + (-4383206.031276241*a^10 + 19523490.2880189*a^8 + 6237485.996192687*a^6 + 166809.9920830212*a^4 - 38026.0974268697*a^2)*nl^16 + (3896183.138912214*a^10 - 15184936.8906813*a^8 - 4158323.997461791*a^6 - 92672.2178239007*a^4 + 16900.48774527542*a^2)*nl^14 + (-2727328.19723855*a^10 + 9110962.134408802*a^8 + 2079161.998730895*a^6 + 37068.8871295603*a^4 - 5070.146323582626*a^2)*nl^12 + (1487633.562130118*a^10 - 4141346.424731274*a^8 - 756058.9086294164*a^6 - 10109.69648988009*a^4 + 921.844786105932*a^2)*nl^10 - 340.5754492056132*a^10 + (-619847.3175542159*a^10 + 1380448.808243758*a^8 + 189014.7271573541*a^6 + 1684.949414980016*a^4 - 76.820398842161*a^2)*nl^8 + ((247.35053257299*a^10 - 2345.488286795025*a^8 + 51280.31023251245*a^6 - 34257.61493040192*a^4 + 7055.849279177906*a^2 - 170.486176417963)*nl^32 + (-3957.608521167841*a^10 + 35182.32430192539*a^8 - 721015.2911640762*a^6 + 462997.8045953113*a^4 - 91388.31410186204*a^2 + 2045.834117015556)*nl^30 + (29682.06390875881*a^10 - 246276.2701134777*a^8 + 4709781.50188326*a^6 - 2901528.501072473*a^4 + 546303.5254464676*a^2 - 11252.08764358556)*nl^28 + (-138516.2982408744*a^10 + 1067197.170491737*a^8 - 18947309.1843446*a^6 + 11174285.0891017*a^4 - 1995682.943033132*a^2 + 37506.95881195187)*nl^26 + (450177.9692828419*a^10 - 3201591.511475211*a^8 + 52456695.5815853*a^6 - 29541754.4782621*a^4 + 4970632.398573037*a^2 - 84390.65732689168)*nl^24 + (-1080427.12627882*a^10 + 7043501.325245466*a^8 - 105757219.942301*a^6 + 56708449.9229891*a^4 - 8913703.391213842*a^2 + 135025.0517230267)*nl^22 + (1980783.064844504*a^10 - 11739168.8754091*a^8 + 160182849.341856*a^6 - 81500086.3341809*a^4 + 11840357.9533283*a^2 - 157529.2270101978)*nl^20 + (-2829690.092635006*a^10 + 15093217.125526*a^8 - 185276141.288415*a^6 + 89071426.038718*a^4 - 11795778.0517048*a^2 + 135025.0517230267)*nl^18 + (3183401.354214381*a^10 - 15093217.125526*a^8 + 164602904.851586*a^6 - 74374909.2335756*a^4 + 8813398.612560969*a^2 - 84390.6573268917)*nl^16 + (-2829690.092635006*a^10 + 11739168.8754091*a^8 - 111945297.655922*a^6 + 47208213.7888486*a^4 - 4877757.60352408*a^2 + 37506.95881195187)*nl^14 + (1980783.064844504*a^10 - 7043501.325245459*a^8 + 57519668.2563666*a^6 - 22416577.3776567*a^4 + 1943673.057805716*a^2 - 11252.08764358556)*nl^12 + (-1080427.12627882*a^10 + 3201591.511475209*a^8 - 21760071.7814454*a^6 + 7719653.767596069*a^4 - 528066.2929641269*a^2 + 2045.834117015556)*nl^10 + 247.35053257299*a^10 + (450177.9692828419*a^10 - 1067197.170491737*a^8 + 5791613.26999893*a^6 - 1821956.213101966*a^4 + 87673.32229990375*a^2 - 170.486176417963)*nl^8 + (-138516.2982408744*a^10 + 246276.2701134777*a^8 - 999200.6029652484*a^6 + 263692.1514315255*a^4 - 6718.122751727153*a^2)*nl^6 + (29682.06390875881*a^10 - 35182.32430192539*a^8 + 94553.58095713923*a^6 - 17648.81050008643*a^4)*nl^4 + (-3957.608521167841*a^10 + 2345.488286795025*a^8 - 3090.947908901912*a^6)*nl^2)*nv^7 + (190722.2515551434*a^10 - 318565.1095947134*a^8 - 29079.18879343909*a^6 - 129.6114934600012*a^4)*nl^6 + ((340.5754492056132*a^10 - 3033.953424711557*a^8 + 2178.822528932337*a^6 - 24430.15170854113*a^4 + 9326.962899799812*a^2 - 234.7413830603392)*nl^32 + (-5449.207187289811*a^10 + 45509.30137067335*a^8 - 34759.42284780213*a^6 + 341892.5124261158*a^4 - 121173.6972985554*a^2 + 2816.896596724071)*nl^30 + (40869.05390467359*a^10 - 318565.1095947134*a^8 + 257855.5543313344*a^6 - 2221458.856062263*a^4 + 726581.2613982793*a^2 - 15492.93128198239)*nl^28 + (-190722.2515551434*a^10 + 1380448.808243758*a^8 - 1180378.977821567*a^6 + 8882465.52541909*a^4 - 2662441.243019164*a^2 + 51643.10427327463)*nl^26 + (619847.3175542159*a^10 - 4141346.424731274*a^8 + 3730151.660622055*a^6 - 24417512.9731201*a^4 + 6651877.98561159*a^2 - 116196.9846148679)*nl^24 + (-1487633.562130118*a^10 + 9110962.134408802*a^8 - 8622166.053114701*a^6 + 48816491.5026754*a^4 - 11965775.1546155*a^2 + 185915.1753837886)*nl^22 + (2727328.19723855*a^10 - 15184936.8906813*a^8 + 15063330.7547681*a^6 - 73196935.588666*a^4 + 15944226.5801735*a^2 - 216901.0379477534)*nl^20 + (-3896183.138912215*a^10 + 19523490.2880189*a^8 - 20258208.9698723*a^6 + 83621867.3409358*a^4 - 15934086.2875263*a^2 + 185915.1753837886)*nl^18 + (4383206.031276242*a^10 - 19523490.2880189*a^8 + 21149278.3978998*a^6 - 73141332.2579717*a^4 + 11942959.4961594*a^2 - 116196.9846148679)*nl^16 + (-3896183.138912215*a^10 + 15184936.8906813*a^8 - 17142492.753499*a^6 + 48742353.7284163*a^4 - 6630752.375929995*a^2 + 51643.10427327463)*nl^14 + (2727328.19723855*a^10 - 9110962.134408802*a^8 + 10701328.0518456*a^6 - 24361909.6424258*a^4 + 2650610.901597471*a^2 - 15492.93128198239)*nl^12 + (-1487633.562130118*a^10 + 4141346.424731274*a^8 - 5053254.750723533*a^6 + 8855506.334779412*a^4 - 722432.9598608026*a^2 + 2816.896596724071)*nl^10 + 340.5754492056132*a^10 + (619847.317554216*a^10 - 1380448.808243758*a^8 + 1747423.159293629*a^6 - 2213034.108987363*a^4 + 120328.6729112916*a^2 - 234.7413830603392)*nl^8 + (-190722.2515551434*a^10 + 318565.1095947134*a^8 - 417791.0926952494*a^6 + 340337.1745045958*a^4 - 9250.14250095765*a^2)*nl^6 + (40869.05390467359*a^10 - 45509.30137067335*a^8 + 61761.52672742413*a^6 - 24300.54021508113*a^4)*nl^4 + (-5449.207187289811*a^10 + 3033.953424711557*a^8 - 4255.907442749414*a^6)*nl^2)*nv^6 + ((-742.0515977189701*a^10 + 7036.464860385076*a^8 - 150749.9827886354*a^6 + 85124.03429111931*a^4 - 14449.42508580657*a^2 + 340.972352835926)*nl^32 + (11872.82556350352*a^10 - 105546.9729057762*a^8 + 2116681.6548587*a^6 - 1141910.066784724*a^4 + 186829.3465331331*a^2 - 4091.668234031113)*nl^30 + (-89046.19172627642*a^10 + 738828.8103404331*a^8 - 13804794.9752151*a^6 + 7098543.747709554*a^4 - 1114897.001704685*a^2 + 22504.17528717112)*nl^28 + (415548.8947226233*a^10 - 3201591.511475213*a^8 + 55435546.2544835*a^6 - 27098688.2452736*a^4 + 4065665.72210543*a^2 - 75013.91762390373)*nl^26 + (-1350533.907848525*a^10 + 9604774.534425633*a^8 - 153150942.849105*a^6 + 70958804.1241997*a^4 - 10108439.4282342*a^2 + 168781.3146537834)*nl^24 + (3241281.378836461*a^10 - 21130503.9757364*a^8 + 307989543.25647*a^6 - 134792431.147794*a^4 + 18094886.1921687*a^2 - 270050.1034460534)*nl^22 + (-5942349.194533512*a^10 + 35217506.6262273*a^8 - 465078353.741515*a^6 + 191500881.070783*a^4 - 23992775.2180211*a^2 + 315058.4540203956)*nl^20 + (8489070.277905017*a^10 - 45279651.376578*a^8 + 535938174.071462*a^6 - 206643560.479857*a^4 + 23859035.5131506*a^2 - 270050.1034460535)*nl^18 + (-9550204.062643144*a^10 + 45279651.376578*a^8 - 473918464.760975*a^6 + 170125349.768967*a^4 - 17793971.8562101*a^2 + 168781.3146537834)*nl^16 + (8489070.277905017*a^10 - 35217506.6262273*a^8 + 320365698.683713*a^6 - 106291722.745373*a^4 + 9829815.043087326*a^2 - 75013.91762390373)*nl^14 + (-5942349.194533512*a^10 + 21130503.9757364*a^8 - 163276888.198667*a^6 + 49583272.8223837*a^4 - 3909636.066423181*a^2 + 22504.17528717112)*nl^12 + (3241281.378836462*a^10 - 9604774.534425627*a^8 + 61061071.4486849*a^6 - 16734794.2807568*a^4 + 1060185.304257663*a^2 - 4091.668234031113)*nl^10 - 742.0515977189701*a^10 + (-1350533.907848526*a^10 + 3201591.511475213*a^8 - 15968458.5114464*a^6 + 3859826.883798034*a^4 - 175684.3711272582*a^2 + 340.972352835926)*nl^8 + (415548.8947226233*a^10 - 738828.8103404331*a^8 + 2673052.278461044*a^6 - 543993.1072933665*a^4 + 13436.24550345431*a^2)*nl^6 + (-89046.19172627643*a^10 + 105546.9729057762*a^8 - 237296.524237889*a^6 + 35297.62100017286*a^4)*nl^4 + (11872.82556350352*a^10 - 7036.464860385076*a^8 + 6181.895817803825*a^6)*nl^2)*nv^5 + (-40869.05390467359*a^10 + 45509.30137067335*a^8 + 2077.084913817077*a^6)*nl^4 + ((-1021.72634761684*a^10 + 9101.860274134671*a^8 - 2280.560144047596*a^6 + 48989.91491054227*a^4 - 18730.74619844178*a^2 + 469.4827661206784)*nl^32 + (16347.62156186943*a^10 - 136527.90411202*a^8 + 40439.65690216516*a^6 - 685469.9742672116*a^4 + 243269.2393832167*a^2 - 5633.793193448141)*nl^30 + (-122607.1617140208*a^10 + 955695.3287841403*a^8 - 326696.3815053147*a^6 + 4453027.408614405*a^4 - 1458232.669120141*a^2 + 30985.86256396477)*nl^28 + (572166.7546654302*a^10 - 4141346.424731275*a^8 + 1604699.047013718*a^6 - 17801999.9379677*a^4 + 5341782.973783603*a^2 - 103286.2085465493)*nl^26 + (-1859541.952662648*a^10 + 12424039.2741938*a^8 - 5381141.322513215*a^6 + 48927698.1640641*a^4 - 13341782.0686501*a^2 + 232393.9692297358)*nl^24 + (4462900.686390355*a^10 - 27332886.4032264*a^8 + 13086008.1087676*a^6 - 97799792.9974339*a^4 + 23992392.065114*a^2 - 371830.3507675772)*nl^22 + (-8181984.591715651*a^10 + 45554810.672044*a^8 - 23889175.5133436*a^6 + 146616284.500109*a^4 - 31959435.2088771*a^2 + 433802.0758955069)*nl^20 + (11688549.4167366*a^10 - 58570470.8640566*a^8 + 33387862.5155243*a^6 - 167466148.004649*a^4 + 31929014.3309356*a^2 - 371830.3507675773)*nl^18 + (-13149618.0938287*a^10 + 58570470.8640566*a^8 - 36061070.7996069*a^6 + 146449474.508026*a^4 - 23923945.0897456*a^2 + 232393.9692297358)*nl^16 + (11688549.4167366*a^10 - 45554810.672044*a^8 + 30126661.5095363*a^6 - 97577379.6746565*a^4 + 13278405.2396053*a^2 - 103286.2085465493)*nl^14 + (-8181984.591715651*a^10 + 27332886.4032264*a^8 - 19323494.1049603*a^6 + 48760888.1719811*a^4 - 5306291.949518524*a^2 + 30985.86256396478)*nl^12 + (4462900.686390355*a^10 - 12424039.2741938*a^8 + 9350450.592817651*a^6 - 17721122.3660487*a^4 + 1445787.764507711*a^2 - 5633.793193448141)*nl^10 - 1021.72634761684*a^10 + (-1859541.952662648*a^10 + 4141346.424731275*a^8 - 3305831.591429905*a^6 + 4427753.167389706*a^4 - 240734.1662214254*a^2 + 469.4827661206784)*nl^8 + (572166.7546654302*a^10 - 955695.3287841403*a^8 + 806502.9965970598*a^6 - 680803.9605026516*a^4 + 18500.2850019153*a^2)*nl^6 + (-122607.1617140208*a^10 + 136527.90411202*a^8 - 121445.9685410312*a^6 + 48601.08043016226*a^4)*nl^4 + (16347.62156186943*a^10 - 9101.860274134671*a^8 + 8511.814885498829*a^6)*nl^2)*nv^4 + ((742.0515977189701*a^10 - 7036.464860385076*a^8 + 147659.0348797335*a^6 - 67475.22379103288*a^4 + 7731.302334079416*a^2 - 170.486176417963)*nl^32 + (-11872.82556350352*a^10 + 105546.9729057762*a^8 - 2070317.436225171*a^6 + 894826.7197835139*a^4 - 99493.75076068015*a^2 + 2045.834117015556)*nl^30 + (89046.19172627642*a^10 - 738828.8103404331*a^8 + 13480245.4447804*a^6 - 5492501.992201689*a^4 + 590883.4270699671*a^2 - 11252.08764358556)*nl^28 + (-415548.8947226233*a^10 + 3201591.511475213*a^8 - 54029164.9559331*a^6 + 20674521.2232422*a^4 - 2144282.615111464*a^2 + 37506.95881195187)*nl^26 + (1350533.907848525*a^10 - 9604774.534425633*a^8 + 148931798.953454*a^6 - 53292344.8136132*a^4 + 5304981.660749285*a^2 - 84390.65732689168)*nl^24 + (-3241281.378836461*a^10 + 21130503.9757364*a^8 - 298707426.686037*a^6 + 99459512.5266211*a^4 - 9448662.210695839*a^2 + 135025.0517230267)*nl^22 + (5942349.194533512*a^10 - 35217506.6262273*a^8 + 449608159.457461*a^6 - 138501503.139024*a^4 + 12464476.5760573*a^2 - 157529.2270101978)*nl^20 + (-8489070.277905017*a^10 + 45279651.376578*a^8 - 516047924.277678*a^6 + 146072842.843561*a^4 - 12330736.8711868*a^2 + 135025.0517230267)*nl^18 + (9550204.062643144*a^10 - 45279651.376578*a^8 + 454028214.967191*a^6 - 117125971.837208*a^4 + 9147747.874737216*a^2 - 84390.6573268917)*nl^16 + (-8489070.277905017*a^10 + 35217506.6262273*a^8 - 304895504.399659*a^6 + 70958804.1241997*a^4 - 5026357.275602412*a^2 + 37506.95881195187)*nl^14 + (5942349.194533513*a^10 - 21130503.9757364*a^8 + 153994771.628235*a^6 - 31916813.5117972*a^4 + 1988252.959429215*a^2 - 11252.08764358556)*nl^12 + (-3241281.378836462*a^10 + 9604774.53442563*a^8 - 56841927.5530338*a^6 + 10310627.2587253*a^4 - 536171.729622945*a^2 + 2045.834117015556)*nl^10 + 742.0515977189701*a^10 + (1350533.907848526*a^10 - 3201591.511475213*a^8 + 14562077.212896*a^6 - 2253785.128290169*a^4 + 88348.77535480524*a^2 - 170.486176417963)*nl^8 + (-415548.8947226233*a^10 + 738828.8103404331*a^8 - 2348502.748026344*a^6 + 296909.7602921565*a^4 - 6718.122751727153*a^2)*nl^6 + (89046.19172627643*a^10 - 105546.9729057762*a^8 + 190932.3056043603*a^6 - 17648.81050008643*a^4)*nl^4 + (-11872.82556350352*a^10 + 7036.464860385076*a^8 - 3090.947908901912*a^6)*nl^2)*nv^3 + (5449.207187289811*a^10 - 3033.953424711557*a^8)*nl^2 + ((1021.72634761684*a^10 - 9101.860274134671*a^8 - 1975.347298701818*a^6 - 24689.37469546113*a^4 + 9480.603697484134*a^2 - 234.7413830603392)*nl^32 + (-16347.62156186943*a^10 + 136527.90411202*a^8 + 23398.95473907605*a^6 + 345262.4112560758*a^4 - 123017.3868707672*a^2 + 2816.896596724071)*nl^30 + (122607.1617140208*a^10 - 955695.3287841403*a^8 - 120173.8999833738*a^6 - 2241678.249042023*a^4 + 736721.5540454446*a^2 - 15492.93128198239)*nl^28 + (-572166.7546654302*a^10 + 4141346.424731275*a^8 + 331738.8394372659*a^6 + 8956603.299678212*a^4 - 2696242.218509714*a^2 + 51643.10427327463)*nl^26 + (1859541.952662648*a^10 - 12424039.2741938*a^8 - 428172.3368397355*a^6 - 24602857.4087679*a^4 + 6727930.180465329*a^2 - 116196.9846148679)*nl^24 + (-4462900.686390355*a^10 + 27332886.4032264*a^8 - 305518.0581911202*a^6 + 49150111.4868415*a^4 - 12087458.6663815*a^2 + 185915.1753837886)*nl^22 + (8181984.591715651*a^10 - 45554810.672044*a^8 + 2588358.762382761*a^6 - 73641762.2342207*a^4 + 16086190.6772338*a^2 - 216901.0379477534)*nl^20 + (-11688549.4167366*a^10 + 58570470.8640566*a^8 - 6001098.121431838*a^6 + 84066693.9864905*a^4 - 16055769.7992923*a^2 + 185915.1753837886)*nl^18 + (13149618.0938287*a^10 - 58570470.8640566*a^8 + 8674306.40551442*a^6 - 73474952.2421377*a^4 + 12019011.6910131*a^2 - 116196.9846148679)*nl^16 + (-11688549.4167366*a^10 + 45554810.672044*a^8 - 8825844.758575443*a^6 + 48927698.1640641*a^4 - 6664553.351420546*a^2 + 51643.10427327463)*nl^14 + (8181984.591715651*a^10 - 27332886.4032264*a^8 + 6543004.054383802*a^6 - 24436047.4166849*a^4 + 2660751.194244636*a^2 - 15492.93128198239)*nl^12 + (-4462900.686390355*a^10 + 12424039.2741938*a^8 - 3541136.933464701*a^6 + 8875725.727759171*a^4 - 724276.6494330144*a^2 + 2816.896596724071)*nl^10 + 1021.72634761684*a^10 + (1859541.952662648*a^10 - 4141346.424731275*a^8 + 1369393.704978921*a^6 - 2216404.007817323*a^4 + 120482.3137089759*a^2 - 234.7413830603392)*nl^8 + (-572166.7546654302*a^10 + 955695.3287841403*a^8 - 359632.7151083713*a^6 + 340596.3974915158*a^4 - 9250.14250095765*a^2)*nl^6 + (122607.1617140208*a^10 - 136527.9041120201*a^8 + 57607.35689978999*a^6 - 24300.54021508113*a^4)*nl^4 + (-16347.62156186943*a^10 + 9101.860274134671*a^8 - 4255.907442749414*a^6)*nl^2)*nv^2 + ((-247.35053257299*a^10 + 2345.488286795025*a^8 - 48189.36232361054*a^6 + 16608.80443031548*a^4 - 337.7265274507546*a^2)*nl^32 + (3957.608521167841*a^10 - 35182.32430192539*a^8 + 674651.0725305476*a^6 - 215914.4575941013*a^4 + 4052.718329409055*a^2)*nl^30 + (-29682.06390875881*a^10 + 246276.2701134777*a^8 - 4385231.971448559*a^6 + 1295486.745564608*a^4 - 22289.9508117498*a^2)*nl^28 + (138516.2982408744*a^10 - 1067197.170491737*a^8 + 17540927.8857942*a^6 - 4750118.067070229*a^4 + 74299.836039166*a^2)*nl^26 + (-450177.9692828418*a^10 + 3201591.511475211*a^8 - 48237551.6859341*a^6 + 11875295.1676756*a^4 - 167174.6310881235*a^2)*nl^24 + (1080427.12627882*a^10 - 7043501.325245462*a^8 + 96475103.3718683*a^6 - 21375531.301816*a^4 + 267479.4097409976*a^2)*nl^22 + (-1980783.064844504*a^10 + 11739168.8754091*a^8 - 144712655.057802*a^6 + 28500708.4024214*a^4 - 312059.3113644972*a^2)*nl^20 + (2829690.092635006*a^10 - 15093217.125526*a^8 + 165385891.494631*a^6 - 28500708.4024214*a^4 + 267479.4097409976*a^2)*nl^18 + (-3183401.354214381*a^10 + 15093217.125526*a^8 - 144712655.057802*a^6 + 21375531.301816*a^4 - 167174.6310881235*a^2)*nl^16 + (2829690.092635006*a^10 - 11739168.8754091*a^8 + 96475103.3718683*a^6 - 11875295.1676756*a^4 + 74299.83603916601*a^2)*nl^14 + (-1980783.064844504*a^10 + 7043501.325245462*a^8 - 48237551.6859341*a^6 + 4750118.067070229*a^4 - 22289.9508117498*a^2)*nl^12 + (1080427.126278821*a^10 - 3201591.511475211*a^8 + 17540927.8857942*a^6 - 1295486.745564608*a^4 + 4052.718329409055*a^2)*nl^10 - 247.35053257299*a^10 + (-450177.9692828419*a^10 + 1067197.170491738*a^8 - 4385231.971448559*a^6 + 215914.4575941013*a^4 - 337.7265274507546*a^2)*nl^8 + (138516.2982408745*a^10 - 246276.2701134777*a^8 + 674651.0725305476*a^6 - 16608.80443031548*a^4)*nl^6 + (-29682.06390875881*a^10 + 35182.32430192539*a^8 - 48189.36232361054*a^6)*nl^4 + (3957.608521167841*a^10 - 2345.488286795025*a^8)*nl^2)*nv + ((1934.29061997787*a^9 - 641.5802369130929*a^7 - 1422.133250031137*a^5 - 341.1774975656799*a^3)*nl^31 + (-29014.35929966805*a^9 + 8982.123316783296*a^7 + 18487.73225040478*a^5 + 4094.12997078816*a^3)*nl^29 + (203100.5150976763*a^9 - 58383.80155909143*a^7 - 110926.3935024287*a^5 - 22517.71483933488*a^3)*nl^27 + (-880102.2320899307*a^9 + 233535.2062363656*a^7 + 406730.1095089052*a^5 + 75059.04946444959*a^3)*nl^25 + (2640306.696269792*a^9 - 642221.8171500053*a^7 - 1016825.273772263*a^5 - 168882.8612950116*a^3)*nl^23 + (-5808674.731793541*a^9 + 1284443.63430001*a^7 + 1830285.492790074*a^5 + 270212.5780720185*a^3)*nl^21 + (9681124.552989237*a^9 - 1926665.451450016*a^7 - 2440380.657053431*a^5 - 315248.0077506883*a^3)*nl^19 + (-12447160.1395576*a^9 + 2201903.373085732*a^7 + 2440380.657053431*a^5 + 270212.5780720185*a^3)*nl^17 + (12447160.1395576*a^9 - 1926665.451450016*a^7 - 1830285.492790074*a^5 - 168882.8612950116*a^3)*nl^15 + (-9681124.552989237*a^9 + 1284443.63430001*a^7 + 1016825.273772263*a^5 + 75059.04946444958*a^3)*nl^13 + (5808674.731793541*a^9 - 642221.8171500053*a^7 - 406730.1095089053*a^5 - 22517.71483933488*a^3)*nl^11 - 1934.29061997787*a^9*nl + (-2640306.696269792*a^9 + 233535.2062363655*a^7 + 110926.3935024287*a^5 + 4094.12997078816*a^3)*nl^9 + (880102.2320899306*a^9 - 58383.80155909141*a^7 - 18487.73225040478*a^5 - 341.1774975656799*a^3)*nl^7 + ((2792.317406342085*a^9 + 28022.70214634566*a^7 - 50114.71581452205*a^5 + 18221.25465116096*a^3 - 1657.960477682254*a)*nl^31 + (-41884.76109513127*a^9 - 392317.8300488393*a^7 + 662933.4963485859*a^5 - 233173.5115237621*a^3 + 19895.52573218704*a)*nl^29 + (293193.3276659189*a^9 + 2550065.895317455*a^7 - 4057696.31341011*a^5 + 1376824.275494591*a^3 - 109425.3915270288*a)*nl^27 + (-1270504.419885649*a^9 - 10200263.5812698*a^7 + 15225299.6022177*a^5 - 4966894.100104233*a^3 + 364751.3050900958*a)*nl^25 + (3811513.259656945*a^9 + 28050724.848492*a^7 - 39104488.3646858*a^5 + 12213581.3084874*a^3 - 820690.4364527158*a)*nl^23 + (-8385329.171245281*a^9 - 56101449.696984*a^7 + 72678805.6465464*a^5 - 21617869.2600856*a^3 + 1313104.698324345*a)*nl^21 + (13975548.6187421*a^9 + 84152174.545476*a^7 - 100722951.845582*a^5 + 28335056.2198586*a^3 - 1531955.481378403*a)*nl^19 + (-17968562.5098113*a^9 - 96173913.7662583*a^7 + 105631651.681535*a^5 - 27846286.759603*a^3 + 1313104.698324345*a)*nl^17 + (17968562.5098113*a^9 + 84152174.545476*a^7 - 84132438.5971054*a^5 + 20518137.9745106*a^3 - 820690.4364527159*a)*nl^15 + (-13975548.6187421*a^9 - 56101449.696984*a^7 + 50558121.3152449*a^5 - 11195311.5996216*a^3 + 364751.3050900958*a)*nl^13 + (8385329.171245279*a^9 + 28050724.848492*a^7 - 22513975.1162098*a^5 + 4396663.063139366*a^3 - 109425.3915270288*a)*nl^11 - 2792.317406342085*a^9*nl + (-3811513.259656945*a^9 - 10200263.5812698*a^7 + 7181414.390835311*a^5 - 1176873.132662755*a^3 + 19895.52573218704*a)*nl^9 + (1270504.419885648*a^9 + 2550065.895317455*a^7 - 1543982.18485313*a^5 + 192442.7231691288*a^3 - 1657.960477682254*a)*nl^7 + (-293193.3276659189*a^9 - 392317.8300488393*a^7 + 198863.1956919126*a^5 - 14518.45570983065*a^3)*nl^5 + (41884.76109513127*a^9 + 28022.70214634566*a^7 - 11442.19075979927*a^5)*nl^3)*nv^7 + ((-1934.29061997787*a^9 + 641.5802369130929*a^7 - 14332.54986052582*a^5 + 20331.55117379083*a^3 - 2282.835733475376*a)*nl^31 + (29014.35929966805*a^9 - 8982.123316783296*a^7 + 202077.8312973925*a^5 - 263968.9877617152*a^3 + 27394.02880170451*a)*nl^29 + (-203100.5150976763*a^9 + 58383.80155909141*a^7 - 1322749.769558254*a^5 + 1581766.861584897*a^3 - 150667.1584093748*a)*nl^27 + (880102.2320899306*a^9 - 233535.2062363656*a^7 + 5327974.542733826*a^5 - 5792305.920864844*a^3 + 502223.8613645827*a)*nl^25 + (-2640306.696269792*a^9 + 642221.8171500053*a^7 - 14753612.5198952*a^5 + 14462000.039796*a^3 - 1130003.688070311*a)*nl^23 + (5808674.731793541*a^9 - 1284443.63430001*a^7 + 29710590.0945449*a^5 - 25997823.4993738*a^3 + 1808005.900912497*a)*nl^21 + (-9681124.552989237*a^9 + 1926665.451450016*a^7 - 44870932.7239491*a^5 + 34618729.2361531*a^3 - 2109340.217731247*a)*nl^19 + (12447160.1395576*a^9 - 2201903.373085732*a^7 + 51629691.778378*a^5 - 34573693.8064744*a^3 + 1808005.900912498*a)*nl^17 + (-12447160.1395576*a^9 + 1926665.451450016*a^7 - 45481027.8882124*a^5 + 25896493.7825968*a^3 - 1130003.688070311*a)*nl^15 + (9681124.552989237*a^9 - 1284443.63430001*a^7 + 30524050.3135628*a^5 - 14368176.2279654*a^3 + 502223.8613645827*a)*nl^13 + (-5808674.731793542*a^9 + 642221.8171500053*a^7 - 15363707.6841586*a^5 + 5739764.586239729*a^3 - 150667.1584093748*a)*nl^11 + 1934.29061997787*a^9*nl + (2640306.696269792*a^9 - 233535.2062363655*a^7 + 5623778.258740302*a^5 - 1563343.27671635*a^3 + 27394.02880170451*a)*nl^9 + (-880102.2320899307*a^9 + 58383.80155909141*a^7 - 1415188.430810278*a^5 + 260216.0352884927*a^3 - 2282.835733475376*a)*nl^7 + (203100.5150976763*a^9 - 8982.123316783298*a^7 + 219143.4302977662*a^5 - 19990.37367622516*a^3)*nl^5 + (-29014.35929966805*a^9 + 641.5802369130929*a^7 - 15754.68311055695*a^5)*nl^3)*nv^6 + (-203100.5150976763*a^9 + 8982.123316783296*a^7 + 1422.133250031137*a^5)*nl^5 + ((-8376.952219026254*a^9 - 84068.10643903699*a^7 + 138901.9566837669*a^5 - 40145.30824365223*a^3 + 3315.920955364508*a)*nl^31 + (125654.2832853938*a^9 + 1176953.490146518*a^7 - 1828609.818408568*a^5 + 510780.610343488*a^3 - 39791.05146437409*a)*nl^29 + (-879579.9829977567*a^9 - 7650197.685952367*a^7 + 11131849.5810886*a^5 - 2998033.281116983*a^3 + 218850.7830540575*a)*nl^27 + (3811513.259656946*a^9 + 30600790.7438095*a^7 - 41510941.370086*a^5 + 10748403.9673011*a^3 - 729502.6101801917*a)*nl^25 + (-11434539.7789708*a^9 - 84152174.545476*a^7 + 105859832.143499*a^5 - 26260048.0929333*a^3 + 1641380.872905432*a)*nl^23 + (25155987.5137358*a^9 + 168304349.090952*a^7 - 195129151.038521*a^5 + 46168355.2817049*a^3 - 2626209.396648691*a)*nl^21 + (-41926645.8562264*a^9 - 252456523.636428*a^7 + 267807956.685067*a^5 - 60091498.6615064*a^3 + 3063910.962756805*a)*nl^19 + (53905687.529434*a^9 + 288521741.298775*a^7 - 277625356.356975*a^5 + 58625190.2807396*a^3 - 2626209.396648691*a)*nl^17 + (-53905687.5294339*a^9 - 252456523.636428*a^7 + 218036416.939639*a^5 - 42869161.4249796*a^3 + 1641380.872905432*a)*nl^15 + (41926645.8562264*a^9 + 168304349.090952*a^7 - 128767098.044617*a^5 + 23205238.9663358*a^3 - 729502.6101801917*a)*nl^13 + (-25155987.5137358*a^9 - 84152174.545476*a^7 + 56088292.3980703*a^5 - 9037710.856406532*a^3 + 218850.7830540575*a)*nl^11 + 8376.952219026254*a^9*nl + (11434539.7789708*a^9 + 30600790.7438095*a^7 - 17379285.735939*a^5 + 2398179.852621472*a^3 - 39791.05146437409*a)*nl^9 + (-3811513.259656945*a^9 - 7650197.685952367*a^7 + 3590707.195417656*a^5 - 388588.2452795878*a^3 + 3315.920955364508*a)*nl^7 + (879579.9829977567*a^9 + 1176953.490146518*a^7 - 436398.916438548*a^5 + 29036.9114196613*a^3)*nl^5 + (-125654.2832853938*a^9 - 84068.10643903699*a^7 + 22884.38151959854*a^5)*nl^3)*nv^5 + ((5802.871859933609*a^9 - 1924.740710739279*a^7 + 27242.9664710205*a^5 - 41004.27984514735*a^3 + 4565.671466950752*a)*nl^31 + (-87043.07789900413*a^9 + 26946.36995034989*a^7 - 385667.9303443803*a^5 + 532032.1054942185*a^3 - 54788.05760340902*a)*nl^29 + (609301.5452930289*a^9 - 175151.4046772742*a^7 + 2534573.14561408*a^5 - 3186051.438009129*a^3 + 301334.3168187496*a)*nl^27 + (-2640306.696269792*a^9 + 700605.6187090968*a^7 - 10249218.9759587*a^5 + 11659670.8911941*a^3 - 1004447.722729165*a)*nl^25 + (7920920.088809375*a^9 - 1926665.451450016*a^7 + 28490399.7660182*a^5 - 29092882.940887*a^3 + 2260007.376140622*a)*nl^23 + (-17426024.1953806*a^9 + 3853330.902900031*a^7 - 57590894.6962998*a^5 + 52265859.5768196*a^3 - 3616011.801824995*a)*nl^21 + (29043373.6589677*a^9 - 5779996.354350047*a^7 + 87301484.7908448*a^5 - 69552706.4800568*a^3 + 4218680.435462494*a)*nl^19 + (-37341480.4186728*a^9 + 6605710.119257197*a^7 - 100819002.899703*a^5 + 69417600.1910208*a^3 - 3616011.801824995*a)*nl^17 + (37341480.4186728*a^9 - 5779996.354350047*a^7 + 89131770.2836348*a^5 - 51961870.4264886*a^3 + 2260007.376140622*a)*nl^15 + (-29043373.6589677*a^9 + 3853330.902900031*a^7 - 60031275.3533533*a^5 + 28811411.5053953*a^3 - 1004447.722729165*a)*nl^13 + (17426024.1953806*a^9 - 1926665.451450016*a^7 + 30320685.2588083*a^5 - 11502046.8873188*a^3 + 301334.3168187496*a)*nl^11 - 5802.871859933609*a^9*nl + (-7920920.088809375*a^9 + 700605.6187090967*a^7 - 11136630.1239782*a^5 + 3130780.683403488*a^3 - 54788.05760340902*a)*nl^9 + (2640306.696269792*a^9 - 175151.4046772742*a^7 + 2811889.129370151*a^5 - 520773.2480745511*a^3 + 4565.671466950752*a)*nl^7 + (-609301.5452930289*a^9 + 26946.36995034989*a^7 - 436864.7273455013*a^5 + 39980.74735245031*a^3)*nl^5 + (87043.07789900413*a^9 - 1924.740710739279*a^7 + 31509.36622111391*a^5)*nl^3)*nv^4 + (29014.35929966805*a^9 - 641.5802369130929*a^7)*nl^3 + ((8376.952219026254*a^9 + 84068.10643903699*a^7 - 127459.7659239676*a^5 + 25626.85253382158*a^3 - 1657.960477682254*a)*nl^31 + (-125654.2832853938*a^9 - 1176953.490146518*a^7 + 1668419.147771378*a^5 - 322040.6861156896*a^3 + 19895.52573218704*a)*nl^29 + (879579.9829977567*a^9 + 7650197.685952367*a^7 - 10090610.2219469*a^5 + 1865593.735750192*a^3 - 109425.3915270288*a)*nl^27 + (-3811513.259656947*a^9 - 30600790.7438095*a^7 + 37345983.9335191*a^5 - 6596125.63428957*a^3 + 364751.3050900958*a)*nl^25 + (11434539.7789708*a^9 + 84152174.545476*a^7 - 94406199.1929394*a^5 + 15879352.2604044*a^3 - 820690.4364527158*a)*nl^23 + (-25155987.5137358*a^9 - 168304349.090952*a^7 + 172221885.137403*a^5 - 27483102.7831529*a^3 + 1313104.698324345*a)*nl^21 + (41926645.8562264*a^9 + 252456523.636428*a^7 - 233447057.83339*a^5 + 35177828.663437*a^3 - 1531955.481378403*a)*nl^19 + (-53905687.5294339*a^9 - 288521741.298775*a^7 + 238355757.669344*a^5 - 33711520.2826702*a^3 + 1313104.698324345*a)*nl^17 + (53905687.529434*a^9 + 252456523.636428*a^7 - 183675518.087962*a^5 + 24183908.9264276*a^3 - 820690.4364527158*a)*nl^15 + (-41926645.8562264*a^9 - 168304349.090952*a^7 + 105859832.143499*a^5 - 12824543.1338069*a^3 + 364751.3050900958*a)*nl^13 + (25155987.5137358*a^9 + 84152174.545476*a^7 - 44634659.4475112*a^5 + 4885432.523394966*a^3 - 109425.3915270288*a)*nl^11 - 8376.952219026254*a^9*nl + (-11434539.7789708*a^9 - 30600790.7438095*a^7 + 13214328.2993721*a^5 - 1265740.307254682*a^3 + 19895.52573218704*a)*nl^9 + (3811513.259656945*a^9 + 7650197.685952367*a^7 - 2549467.836275922*a^5 + 199848.3210517894*a^3 - 1657.960477682254*a)*nl^7 + (-879579.9829977567*a^9 - 1176953.490146518*a^7 + 276208.2458013581*a^5 - 14518.45570983065*a^3)*nl^5 + (125654.2832853938*a^9 + 84068.10643903699*a^7 - 11442.19075979927*a^5)*nl^3)*nv^3 + ((-5802.871859933609*a^9 + 1924.740710739279*a^7 - 11488.28336046355*a^5 + 21013.90616892219*a^3 - 2282.835733475376*a)*nl^31 + (87043.07789900413*a^9 - 26946.36995034989*a^7 + 165102.366796583*a^5 - 272157.2477032915*a^3 + 27394.02880170451*a)*nl^29 + (-609301.5452930289*a^9 + 175151.4046772743*a^7 - 1100896.982553397*a^5 + 1626802.291263567*a^3 - 150667.1584093748*a)*nl^27 + (2640306.696269792*a^9 - 700605.6187090968*a^7 + 4514514.323716015*a^5 - 5942424.019793743*a^3 + 502223.8613645827*a)*nl^25 + (-7920920.088809375*a^9 + 1926665.451450016*a^7 - 12719961.9723507*a^5 + 14799765.762386*a^3 - 1130003.688070311*a)*nl^23 + (17426024.1953806*a^9 - 3853330.902900031*a^7 + 26050019.1089648*a^5 - 26538248.6555178*a^3 + 1808005.900912497*a)*nl^21 + (-29043373.6589677*a^9 + 5779996.354350047*a^7 - 39990171.4098422*a^5 + 35249225.2516544*a^3 - 2109340.217731247*a)*nl^19 + (37341480.4186728*a^9 - 6605710.119257197*a^7 + 46748930.4642712*a^5 - 35114118.9626184*a^3 + 1808005.900912498*a)*nl^17 + (-37341480.4186728*a^9 + 5779996.354350047*a^7 - 41820456.9026323*a^5 + 26234259.5051868*a^3 - 1130003.688070311*a)*nl^15 + (29043373.6589677*a^9 - 3853330.902900031*a^7 + 28490399.7660182*a^5 - 14518294.3268943*a^3 + 502223.8613645827*a)*nl^13 + (-17426024.1953806*a^9 + 1926665.451450016*a^7 - 14550247.4651408*a^5 + 5784800.015918399*a^3 - 150667.1584093748*a)*nl^11 + 5802.871859933609*a^9*nl + (7920920.088809375*a^9 - 700605.6187090967*a^7 + 5401925.471735444*a^5 - 1571531.536657927*a^3 + 27394.02880170451*a)*nl^9 + (-2640306.696269792*a^9 + 175151.4046772742*a^7 - 1378212.966309468*a^5 + 260898.3902836241*a^3 - 2282.835733475376*a)*nl^7 + (609301.5452930289*a^9 - 26946.36995034989*a^7 + 216299.1637977039*a^5 - 19990.37367622516*a^3)*nl^5 + (-87043.07789900413*a^9 + 1924.740710739279*a^7 - 15754.68311055695*a^5)*nl^3)*nv^2 + ((-2792.317406342085*a^9 - 28022.70214634566*a^7 + 38672.52505472278*a^5 - 3702.79894133031*a^3)*nl^31 + (41884.76109513127*a^9 + 392317.8300488393*a^7 - 502742.8257113961*a^5 + 44433.58729596373*a^3)*nl^29 + (-293193.3276659189*a^9 - 2550065.895317456*a^7 + 3016456.954268376*a^5 - 244384.7301278005*a^3)*nl^27 + (1270504.419885649*a^9 + 10200263.5812698*a^7 - 11060342.1656507*a^5 + 814615.7670926682*a^3)*nl^25 + (-3811513.259656945*a^9 - 28050724.848492*a^7 + 27650855.4141268*a^5 - 1832885.475958503*a^3)*nl^23 + (8385329.171245281*a^9 + 56101449.696984*a^7 - 49771539.7454282*a^5 + 2932616.761533606*a^3)*nl^21 + (-13975548.6187421*a^9 - 84152174.545476*a^7 + 66362052.9939043*a^5 - 3421386.221789206*a^3)*nl^19 + (17968562.5098113*a^9 + 96173913.7662583*a^7 - 66362052.9939043*a^5 + 2932616.761533606*a^3)*nl^17 + (-17968562.5098113*a^9 - 84152174.545476*a^7 + 49771539.7454282*a^5 - 1832885.475958503*a^3)*nl^15 + (13975548.6187421*a^9 + 56101449.696984*a^7 - 27650855.4141268*a^5 + 814615.7670926682*a^3)*nl^13 + (-8385329.171245281*a^9 - 28050724.848492*a^7 + 11060342.1656507*a^5 - 244384.7301278005*a^3)*nl^11 + 2792.317406342085*a^9*nl + (3811513.259656945*a^9 + 10200263.5812698*a^7 - 3016456.954268376*a^5 + 44433.58729596373*a^3)*nl^9 + (-1270504.419885649*a^9 - 2550065.895317456*a^7 + 502742.8257113961*a^5 - 3702.79894133031*a^3)*nl^7 + (293193.3276659189*a^9 + 392317.8300488393*a^7 - 38672.52505472278*a^5)*nl^5 + (-41884.76109513127*a^9 - 28022.70214634566*a^7)*nl^3)*nv)*sqrt(-1.0*nl^2 + 1.0) + (((-6634.012962169122*a^7 + 58248.2683574047*a^5 - 21171.26556842952*a^3 + 480.11579705128*a)*nl^32 + (99510.19443253685*a^7 - 815475.7570036658*a^5 + 275226.4523895838*a^3 - 5761.38956461536*a)*nl^30 + (-696571.3610277579*a^7 + 5300592.420523828*a^5 - 1651358.714337503*a^3 + 31687.64260538448*a)*nl^28 + (3018475.897786951*a^7 - 21202369.6820953*a^5 + 6054981.952570844*a^3 - 105625.4753512816*a)*nl^26 + (-9055427.693360854*a^7 + 58306516.6257621*a^5 - 15137454.8814271*a^3 + 237657.3195403836*a)*nl^24 + (19921940.9253939*a^7 - 116613033.251524*a^5 + 27247418.7865688*a^3 - 380251.7112646137*a)*nl^22 + (-33203234.8756565*a^7 + 174919549.877286*a^5 - 36329891.7154251*a^3 + 443626.9964753826*a)*nl^20 + (42689873.4115583*a^7 - 199908057.002613*a^5 + 36329891.7154251*a^3 - 380251.7112646137*a)*nl^18 + (-42689873.4115583*a^7 + 174919549.877286*a^5 - 27247418.7865688*a^3 + 237657.3195403836*a)*nl^16 + (33203234.8756565*a^7 - 116613033.251524*a^5 + 15137454.8814271*a^3 - 105625.4753512816*a)*nl^14 + (-19921940.9253939*a^7 + 58306516.6257621*a^5 - 6054981.952570844*a^3 + 31687.64260538448*a)*nl^12 + (9055427.693360852*a^7 - 21202369.6820953*a^5 + 1651358.714337503*a^3 - 5761.38956461536*a)*nl^10 + 6634.012962169122*a^7*nl^2 + (-3018475.897786951*a^7 + 5300592.420523828*a^5 - 275226.4523895838*a^3 + 480.11579705128*a)*nl^8 + (696571.3610277579*a^7 - 815475.7570036658*a^5 + 21171.26556842952*a^3)*nl^6 + (-99510.19443253685*a^7 + 58248.2683574047*a^5)*nl^4)*nv^6 + ((11294.48766992642*a^7 - 36443.55577869599*a^5 + 15251.10565494534*a^3 - 465.7157104548775*a)*nl^32 + (-169417.3150488964*a^7 + 510209.7809017438*a^5 - 198264.3735142894*a^3 + 5588.58852545853*a)*nl^30 + (1185921.205342275*a^7 - 3316363.575861335*a^5 + 1189586.241085737*a^3 - 30737.23689002192*a)*nl^28 + (-5138991.889816523*a^7 + 13265454.3034453*a^5 - 4361816.217314367*a^3 + 102457.456300073*a)*nl^26 + (15416975.6694496*a^7 - 36479999.3344747*a^5 + 10904540.5432859*a^3 - 230529.2766751644*a)*nl^24 + (-33917346.472789*a^7 + 72959998.6689494*a^5 - 19628172.9779146*a^3 + 368846.842680263*a)*nl^22 + (56528910.7879817*a^7 - 109439998.003424*a^5 + 26170897.3038862*a^3 - 430321.3164603068*a)*nl^20 + (-72680028.1559765*a^7 + 125074283.432485*a^5 - 26170897.3038862*a^3 + 368846.842680263*a)*nl^18 + (72680028.1559765*a^7 - 109439998.003424*a^5 + 19628172.9779146*a^3 - 230529.2766751644*a)*nl^16 + (-56528910.7879817*a^7 + 72959998.6689494*a^5 - 10904540.5432859*a^3 + 102457.4563000731*a)*nl^14 + (33917346.472789*a^7 - 36479999.3344747*a^5 + 4361816.217314368*a^3 - 30737.23689002192*a)*nl^12 + (-15416975.6694496*a^7 + 13265454.3034453*a^5 - 1189586.241085737*a^3 + 5588.58852545853*a)*nl^10 - 11294.48766992642*a^7*nl^2 + (5138991.889816523*a^7 - 3316363.575861334*a^5 + 198264.3735142894*a^3 - 465.7157104548775*a)*nl^8 + (-1185921.205342274*a^7 + 510209.7809017438*a^5 - 15251.10565494534*a^3)*nl^6 + (169417.3150488964*a^7 - 36443.55577869599*a^5)*nl^4)*nv^5 + ((13268.02592433824*a^7 - 116496.5367148094*a^5 + 42342.53113685905*a^3 - 960.2315941025599*a)*nl^32 + (-199020.3888650737*a^7 + 1630951.514007332*a^5 - 550452.9047791676*a^3 + 11522.77912923072*a)*nl^30 + (1393142.722055516*a^7 - 10601184.8410477*a^5 + 3302717.428675006*a^3 - 63375.28521076895*a)*nl^28 + (-6036951.795573902*a^7 + 42404739.3641906*a^5 - 12109963.9051417*a^3 + 211250.9507025632*a)*nl^26 + (18110855.3867217*a^7 - 116613033.251524*a^5 + 30274909.7628542*a^3 - 475314.6390807671*a)*nl^24 + (-39843881.8507877*a^7 + 233226066.503048*a^5 - 54494837.5731376*a^3 + 760503.4225292273*a)*nl^22 + (66406469.7513129*a^7 - 349839099.754573*a^5 + 72659783.4308501*a^3 - 887253.9929507653*a)*nl^20 + (-85379746.8231166*a^7 + 399816114.005226*a^5 - 72659783.4308501*a^3 + 760503.4225292273*a)*nl^18 + (85379746.8231166*a^7 - 349839099.754573*a^5 + 54494837.5731376*a^3 - 475314.6390807671*a)*nl^16 + (-66406469.7513129*a^7 + 233226066.503048*a^5 - 30274909.7628542*a^3 + 211250.9507025632*a)*nl^14 + (39843881.8507877*a^7 - 116613033.251524*a^5 + 12109963.9051417*a^3 - 63375.28521076895*a)*nl^12 + (-18110855.3867217*a^7 + 42404739.3641906*a^5 - 3302717.428675006*a^3 + 11522.77912923072*a)*nl^10 - 13268.02592433824*a^7*nl^2 + (6036951.795573902*a^7 - 10601184.8410477*a^5 + 550452.9047791676*a^3 - 960.2315941025599*a)*nl^8 + (-1393142.722055516*a^7 + 1630951.514007332*a^5 - 42342.53113685905*a^3)*nl^6 + (199020.3888650737*a^7 - 116496.5367148094*a^5)*nl^4)*nv^4 + ((-22588.97533985285*a^7 + 72887.11155739198*a^5 - 30502.21130989068*a^3 + 931.431420909755*a)*nl^32 + (338834.6300977927*a^7 - 1020419.561803488*a^5 + 396528.7470285788*a^3 - 11177.17705091706*a)*nl^30 + (-2371842.410684549*a^7 + 6632727.15172267*a^5 - 2379172.482171473*a^3 + 61474.47378004383*a)*nl^28 + (10277983.779633*a^7 - 26530908.6068907*a^5 + 8723632.434628734*a^3 - 204914.9126001461*a)*nl^26 + (-30833951.3388991*a^7 + 72959998.6689494*a^5 - 21809081.0865718*a^3 + 461058.5533503287*a)*nl^24 + (67834692.9455781*a^7 - 145919997.337899*a^5 + 39256345.9558293*a^3 - 737693.685360526*a)*nl^22 + (-113057821.575963*a^7 + 218879996.006848*a^5 - 52341794.6077724*a^3 + 860642.6329206135*a)*nl^20 + (145360056.311953*a^7 - 250148566.864969*a^5 + 52341794.6077724*a^3 - 737693.685360526*a)*nl^18 + (-145360056.311953*a^7 + 218879996.006848*a^5 - 39256345.9558293*a^3 + 461058.5533503287*a)*nl^16 + (113057821.575963*a^7 - 145919997.337899*a^5 + 21809081.0865718*a^3 - 204914.9126001461*a)*nl^14 + (-67834692.9455781*a^7 + 72959998.6689494*a^5 - 8723632.434628734*a^3 + 61474.47378004383*a)*nl^12 + (30833951.3388991*a^7 - 26530908.6068907*a^5 + 2379172.482171473*a^3 - 11177.17705091706*a)*nl^10 + 22588.97533985285*a^7*nl^2 + (-10277983.779633*a^7 + 6632727.15172267*a^5 - 396528.7470285788*a^3 + 931.431420909755*a)*nl^8 + (2371842.410684549*a^7 - 1020419.561803488*a^5 + 30502.21130989068*a^3)*nl^6 + (-338834.6300977927*a^7 + 72887.11155739198*a^5)*nl^4)*nv^3 + ((-6634.012962169122*a^7 + 58248.2683574047*a^5 - 21171.26556842952*a^3 + 480.11579705128*a)*nl^32 + (99510.19443253685*a^7 - 815475.7570036658*a^5 + 275226.4523895838*a^3 - 5761.38956461536*a)*nl^30 + (-696571.3610277579*a^7 + 5300592.420523828*a^5 - 1651358.714337503*a^3 + 31687.64260538448*a)*nl^28 + (3018475.897786951*a^7 - 21202369.6820953*a^5 + 6054981.952570844*a^3 - 105625.4753512816*a)*nl^26 + (-9055427.693360852*a^7 + 58306516.6257621*a^5 - 15137454.8814271*a^3 + 237657.3195403836*a)*nl^24 + (19921940.9253939*a^7 - 116613033.251524*a^5 + 27247418.7865688*a^3 - 380251.7112646137*a)*nl^22 + (-33203234.8756565*a^7 + 174919549.877286*a^5 - 36329891.7154251*a^3 + 443626.9964753826*a)*nl^20 + (42689873.4115583*a^7 - 199908057.002613*a^5 + 36329891.7154251*a^3 - 380251.7112646137*a)*nl^18 + (-42689873.4115583*a^7 + 174919549.877286*a^5 - 27247418.7865688*a^3 + 237657.3195403836*a)*nl^16 + (33203234.8756565*a^7 - 116613033.251524*a^5 + 15137454.8814271*a^3 - 105625.4753512816*a)*nl^14 + (-19921940.9253939*a^7 + 58306516.6257621*a^5 - 6054981.952570844*a^3 + 31687.64260538448*a)*nl^12 + (9055427.693360854*a^7 - 21202369.6820953*a^5 + 1651358.714337503*a^3 - 5761.38956461536*a)*nl^10 + 6634.012962169122*a^7*nl^2 + (-3018475.897786951*a^7 + 5300592.420523828*a^5 - 275226.4523895838*a^3 + 480.11579705128*a)*nl^8 + (696571.3610277578*a^7 - 815475.7570036658*a^5 + 21171.26556842952*a^3)*nl^6 + (-99510.19443253685*a^7 + 58248.2683574047*a^5)*nl^4)*nv^2 + ((11294.48766992642*a^7 - 36443.55577869599*a^5 + 15251.10565494534*a^3 - 465.7157104548775*a)*nl^32 + (-169417.3150488964*a^7 + 510209.7809017438*a^5 - 198264.3735142894*a^3 + 5588.58852545853*a)*nl^30 + (1185921.205342274*a^7 - 3316363.575861335*a^5 + 1189586.241085737*a^3 - 30737.23689002192*a)*nl^28 + (-5138991.889816523*a^7 + 13265454.3034453*a^5 - 4361816.217314368*a^3 + 102457.456300073*a)*nl^26 + (15416975.6694496*a^7 - 36479999.3344747*a^5 + 10904540.5432859*a^3 - 230529.2766751644*a)*nl^24 + (-33917346.472789*a^7 + 72959998.6689494*a^5 - 19628172.9779146*a^3 + 368846.842680263*a)*nl^22 + (56528910.7879817*a^7 - 109439998.003424*a^5 + 26170897.3038862*a^3 - 430321.3164603068*a)*nl^20 + (-72680028.1559765*a^7 + 125074283.432485*a^5 - 26170897.3038862*a^3 + 368846.842680263*a)*nl^18 + (72680028.1559765*a^7 - 109439998.003424*a^5 + 19628172.9779146*a^3 - 230529.2766751644*a)*nl^16 + (-56528910.7879817*a^7 + 72959998.6689494*a^5 - 10904540.5432859*a^3 + 102457.4563000731*a)*nl^14 + (33917346.472789*a^7 - 36479999.3344747*a^5 + 4361816.217314367*a^3 - 30737.23689002192*a)*nl^12 + (-15416975.6694496*a^7 + 13265454.3034453*a^5 - 1189586.241085737*a^3 + 5588.58852545853*a)*nl^10 - 11294.48766992642*a^7*nl^2 + (5138991.889816523*a^7 - 3316363.575861335*a^5 + 198264.3735142894*a^3 - 465.7157104548775*a)*nl^8 + (-1185921.205342275*a^7 + 510209.7809017438*a^5 - 15251.10565494534*a^3)*nl^6 + (169417.3150488964*a^7 - 36443.55577869599*a^5)*nl^4)*nv + (((1748.76826529104*a^8 + 35820.7217794803*a^6 - 47426.15068109697*a^4 + 4966.249899464528*a^2)*nl^31 + (-26231.52397936559*a^8 - 501490.1049127242*a^6 + 616539.9588542606*a^4 - 59594.99879357434*a^2)*nl^29 + (183620.6678555591*a^8 + 3259685.681932708*a^6 - 3699239.753125563*a^4 + 327772.4933646589*a^2)*nl^27 + (-795689.560707423*a^8 - 13038742.7277308*a^6 + 13563879.0947937*a^4 - 1092574.977882196*a^2)*nl^25 + (2387068.682122269*a^8 + 35856542.5012598*a^6 - 33909697.7369843*a^4 + 2458293.700234942*a^2)*nl^23 + (-5251551.100668993*a^8 - 71713085.0025196*a^6 + 61037455.9265718*a^4 - 3933269.920375907*a^2)*nl^21 + (8752585.167781653*a^8 + 107569627.503779*a^6 - 81383274.5687624*a^4 + 4588814.907105224*a^2)*nl^19 + (-11253323.7871478*a^8 - 122936717.147176*a^6 + 81383274.5687624*a^4 - 3933269.920375907*a^2)*nl^17 + (11253323.7871478*a^8 + 107569627.503779*a^6 - 61037455.9265718*a^4 + 2458293.700234942*a^2)*nl^15 + (-8752585.167781655*a^8 - 71713085.0025196*a^6 + 33909697.7369843*a^4 - 1092574.977882196*a^2)*nl^13 + (5251551.100668991*a^8 + 35856542.5012598*a^6 - 13563879.0947937*a^4 + 327772.4933646589*a^2)*nl^11 + (-2387068.682122269*a^8 - 13038742.7277308*a^6 + 3699239.753125564*a^4 - 59594.99879357434*a^2)*nl^9 - 1748.76826529104*a^8*nl + (795689.560707423*a^8 + 3259685.681932707*a^6 - 616539.9588542606*a^4 + 4966.249899464528*a^2)*nl^7 + (-183620.6678555591*a^8 - 501490.1049127242*a^6 + 47426.15068109697*a^4)*nl^5 + (26231.52397936559*a^8 + 35820.7217794803*a^6)*nl^3)*nv^6 + ((2407.868425883686*a^8 - 26302.86255025803*a^6 + 30655.1945279348*a^4 - 4119.861118470091*a^2)*nl^31 + (-36118.02638825528*a^8 + 368240.0757036123*a^6 - 398517.5288631524*a^4 + 49438.33342164109*a^2)*nl^29 + (252826.184717787*a^8 - 2393560.49207348*a^6 + 2391105.173178914*a^4 - 271910.833819026*a^2)*nl^27 + (-1095580.133777077*a^8 + 9574241.96829392*a^6 - 8767385.634989353*a^4 + 906369.4460634199*a^2)*nl^25 + (3286740.401331231*a^8 - 26329165.4128083*a^6 + 21918464.0874734*a^4 - 2039331.253642695*a^2)*nl^23 + (-7230828.882928707*a^8 + 52658330.8256165*a^6 - 39453235.3574521*a^4 + 3262930.005828312*a^2)*nl^21 + (12051381.4715479*a^8 - 78987496.2384248*a^6 + 52604313.8099361*a^4 - 3806751.673466364*a^2)*nl^19 + (-15494633.3205615*a^8 + 90271424.2724855*a^6 - 52604313.8099361*a^4 + 3262930.005828312*a^2)*nl^17 + (15494633.3205615*a^8 - 78987496.2384248*a^6 + 39453235.3574521*a^4 - 2039331.253642695*a^2)*nl^15 + (-12051381.4715478*a^8 + 52658330.8256165*a^6 - 21918464.0874734*a^4 + 906369.44606342*a^2)*nl^13 + (7230828.882928707*a^8 - 26329165.4128083*a^6 + 8767385.634989353*a^4 - 271910.833819026*a^2)*nl^11 + (-3286740.40133123*a^8 + 9574241.96829392*a^6 - 2391105.173178914*a^4 + 49438.33342164109*a^2)*nl^9 - 2407.868425883686*a^8*nl + (1095580.133777077*a^8 - 2393560.49207348*a^6 + 398517.5288631525*a^4 - 4119.861118470091*a^2)*nl^7 + (-252826.184717787*a^8 + 368240.0757036123*a^6 - 30655.1945279348*a^4)*nl^5 + (36118.02638825528*a^8 - 26302.86255025803*a^6)*nl^3)*nv^5 + ((-3497.536530582079*a^8 - 71641.4435589606*a^6 + 94852.30136219395*a^4 - 9932.499798929057*a^2)*nl^31 + (52463.04795873119*a^8 + 1002980.209825448*a^6 - 1233079.917708521*a^4 + 119189.9975871487*a^2)*nl^29 + (-367241.3357111183*a^8 - 6519371.363865415*a^6 + 7398479.506251127*a^4 - 655544.9867293178*a^2)*nl^27 + (1591379.121414846*a^8 + 26077485.4554617*a^6 - 27127758.1895875*a^4 + 2185149.955764392*a^2)*nl^25 + (-4774137.364244537*a^8 - 71713085.0025196*a^6 + 67819395.4739687*a^4 - 4916587.400469883*a^2)*nl^23 + (10503102.201338*a^8 + 143426170.005039*a^6 - 122074911.853144*a^4 + 7866539.840751814*a^2)*nl^21 + (-17505170.3355633*a^8 - 215139255.007559*a^6 + 162766549.137525*a^4 - 9177629.814210448*a^2)*nl^19 + (22506647.5742957*a^8 + 245873434.294353*a^6 - 162766549.137525*a^4 + 7866539.840751814*a^2)*nl^17 + (-22506647.5742957*a^8 - 215139255.007559*a^6 + 122074911.853144*a^4 - 4916587.400469883*a^2)*nl^15 + (17505170.3355633*a^8 + 143426170.005039*a^6 - 67819395.4739687*a^4 + 2185149.955764392*a^2)*nl^13 + (-10503102.201338*a^8 - 71713085.0025196*a^6 + 27127758.1895875*a^4 - 655544.9867293178*a^2)*nl^11 + (4774137.364244537*a^8 + 26077485.4554617*a^6 - 7398479.506251127*a^4 + 119189.9975871487*a^2)*nl^9 + 3497.536530582079*a^8*nl + (-1591379.121414846*a^8 - 6519371.363865415*a^6 + 1233079.917708521*a^4 - 9932.499798929057*a^2)*nl^7 + (367241.3357111183*a^8 + 1002980.209825448*a^6 - 94852.30136219395*a^4)*nl^5 + (-52463.04795873119*a^8 - 71641.4435589606*a^6)*nl^3)*nv^4 + ((-4815.736851767371*a^8 + 52605.72510051605*a^6 - 61310.3890558696*a^4 + 8239.722236940182*a^2)*nl^31 + (72236.05277651057*a^8 - 736480.1514072246*a^6 + 797035.0577263049*a^4 - 98876.66684328219*a^2)*nl^29 + (-505652.3694355739*a^8 + 4787120.98414696*a^6 - 4782210.346357829*a^4 + 543821.667638052*a^2)*nl^27 + (2191160.267554154*a^8 - 19148483.9365878*a^6 + 17534771.2699787*a^4 - 1812738.89212684*a^2)*nl^25 + (-6573480.802662461*a^8 + 52658330.8256165*a^6 - 43836928.1749468*a^4 + 4078662.50728539*a^2)*nl^23 + (14461657.7658574*a^8 - 105316661.651233*a^6 + 78906470.7149042*a^4 - 6525860.011656624*a^2)*nl^21 + (-24102762.9430957*a^8 + 157974992.47685*a^6 - 105208627.619872*a^4 + 7613503.346932728*a^2)*nl^19 + (30989266.641123*a^8 - 180542848.544971*a^6 + 105208627.619872*a^4 - 6525860.011656624*a^2)*nl^17 + (-30989266.641123*a^8 + 157974992.47685*a^6 - 78906470.7149042*a^4 + 4078662.50728539*a^2)*nl^15 + (24102762.9430957*a^8 - 105316661.651233*a^6 + 43836928.1749468*a^4 - 1812738.89212684*a^2)*nl^13 + (-14461657.7658574*a^8 + 52658330.8256165*a^6 - 17534771.2699787*a^4 + 543821.667638052*a^2)*nl^11 + (6573480.80266246*a^8 - 19148483.9365878*a^6 + 4782210.346357829*a^4 - 98876.66684328219*a^2)*nl^9 + 4815.736851767371*a^8*nl + (-2191160.267554154*a^8 + 4787120.98414696*a^6 - 797035.0577263049*a^4 + 8239.722236940182*a^2)*nl^7 + (505652.3694355739*a^8 - 736480.1514072246*a^6 + 61310.3890558696*a^4)*nl^5 + (-72236.05277651057*a^8 + 52605.72510051605*a^6)*nl^3)*nv^3 + ((1748.76826529104*a^8 + 35820.7217794803*a^6 - 47426.15068109697*a^4 + 4966.249899464528*a^2)*nl^31 + (-26231.52397936559*a^8 - 501490.1049127242*a^6 + 616539.9588542606*a^4 - 59594.99879357434*a^2)*nl^29 + (183620.6678555591*a^8 + 3259685.681932707*a^6 - 3699239.753125563*a^4 + 327772.4933646589*a^2)*nl^27 + (-795689.560707423*a^8 - 13038742.7277308*a^6 + 13563879.0947937*a^4 - 1092574.977882196*a^2)*nl^25 + (2387068.682122269*a^8 + 35856542.5012598*a^6 - 33909697.7369843*a^4 + 2458293.700234942*a^2)*nl^23 + (-5251551.100668992*a^8 - 71713085.0025196*a^6 + 61037455.9265718*a^4 - 3933269.920375907*a^2)*nl^21 + (8752585.167781655*a^8 + 107569627.503779*a^6 - 81383274.5687624*a^4 + 4588814.907105224*a^2)*nl^19 + (-11253323.7871478*a^8 - 122936717.147176*a^6 + 81383274.5687624*a^4 - 3933269.920375907*a^2)*nl^17 + (11253323.7871478*a^8 + 107569627.503779*a^6 - 61037455.9265718*a^4 + 2458293.700234942*a^2)*nl^15 + (-8752585.167781653*a^8 - 71713085.0025196*a^6 + 33909697.7369843*a^4 - 1092574.977882196*a^2)*nl^13 + (5251551.100668992*a^8 + 35856542.5012598*a^6 - 13563879.0947937*a^4 + 327772.4933646589*a^2)*nl^11 + (-2387068.682122269*a^8 - 13038742.7277308*a^6 + 3699239.753125564*a^4 - 59594.99879357434*a^2)*nl^9 - 1748.76826529104*a^8*nl + (795689.560707423*a^8 + 3259685.681932707*a^6 - 616539.9588542606*a^4 + 4966.249899464528*a^2)*nl^7 + (-183620.6678555591*a^8 - 501490.1049127242*a^6 + 47426.15068109697*a^4)*nl^5 + (26231.52397936559*a^8 + 35820.7217794803*a^6)*nl^3)*nv^2 + ((2407.868425883686*a^8 - 26302.86255025803*a^6 + 30655.1945279348*a^4 - 4119.861118470091*a^2)*nl^31 + (-36118.02638825528*a^8 + 368240.0757036123*a^6 - 398517.5288631524*a^4 + 49438.33342164109*a^2)*nl^29 + (252826.184717787*a^8 - 2393560.49207348*a^6 + 2391105.173178914*a^4 - 271910.833819026*a^2)*nl^27 + (-1095580.133777077*a^8 + 9574241.96829392*a^6 - 8767385.634989355*a^4 + 906369.4460634199*a^2)*nl^25 + (3286740.401331231*a^8 - 26329165.4128083*a^6 + 21918464.0874734*a^4 - 2039331.253642695*a^2)*nl^23 + (-7230828.882928708*a^8 + 52658330.8256165*a^6 - 39453235.3574521*a^4 + 3262930.005828313*a^2)*nl^21 + (12051381.4715478*a^8 - 78987496.2384248*a^6 + 52604313.8099361*a^4 - 3806751.673466364*a^2)*nl^19 + (-15494633.3205615*a^8 + 90271424.2724855*a^6 - 52604313.8099361*a^4 + 3262930.005828312*a^2)*nl^17 + (15494633.3205615*a^8 - 78987496.2384248*a^6 + 39453235.3574521*a^4 - 2039331.253642695*a^2)*nl^15 + (-12051381.4715478*a^8 + 52658330.8256165*a^6 - 21918464.0874734*a^4 + 906369.44606342*a^2)*nl^13 + (7230828.882928707*a^8 - 26329165.4128083*a^6 + 8767385.634989353*a^4 - 271910.833819026*a^2)*nl^11 + (-3286740.401331231*a^8 + 9574241.96829392*a^6 - 2391105.173178914*a^4 + 49438.33342164109*a^2)*nl^9 - 2407.868425883686*a^8*nl + (1095580.133777077*a^8 - 2393560.49207348*a^6 + 398517.5288631524*a^4 - 4119.861118470091*a^2)*nl^7 + (-252826.184717787*a^8 + 368240.0757036123*a^6 - 30655.1945279348*a^4)*nl^5 + (36118.02638825528*a^8 - 26302.86255025803*a^6)*nl^3)*nv)*sqrt(-1.0*nl^2 + 1.0))*sqrt(-1.0*nv^2 + 1.0))
```
https://ask.sagemath.org/question/8403/full-simplify-sage-vs-mathematica/
might want to try with mathematica but it doesnt let me create an account/download trial. Anyway i just wonder where i got other smiths from because im sure i didnt use any siplifying tool for it yet we have those expressions XD

Reply:
> i got other smiths from because im sure i didnt use any siplifying tool for it yet we have those expressions XD

I simplified for you